<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />


<meta name="author" content="Akhilesh" />

<meta name="date" content="2022-11-14" />

<title>MSK_Impact_result</title>

<script src="site_libs/header-attrs-2.13/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/font-awesome-5.1.0/css/all.css" rel="stylesheet" />
<link href="site_libs/font-awesome-5.1.0/css/v4-shims.css" rel="stylesheet" />

<link rel="icon" href="https://github.com/workflowr/workflowr-assets/raw/main/img/reproducible.png">
<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>



<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>









<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
details > summary > p:only-child {
  display: inline;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.tab('show');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-bs-toggle="collapse" data-target="#navbar" data-bs-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">myproject</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/Akhileshpandey87/myproject">
    <span class="fab fa-github"></span>
     
    Source code
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">



<h1 class="title toc-ignore">MSK_Impact_result</h1>
<h4 class="author">Akhilesh</h4>
<h4 class="date">2022-11-14</h4>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span>
workflowr <span class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span
class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span> </a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2022-11-17
</p>
<p>
<strong>Checks:</strong> <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 6
<span class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span> 1
</p>
<p>
<strong>Knit directory:</strong> <code>myproject/</code> <span
class="glyphicon glyphicon-question-sign" aria-hidden="true"
title="This is the local directory in which the code in this file was executed.">
</span>
</p>
<p>
This reproducible <a href="https://rmarkdown.rstudio.com">R Markdown</a>
analysis was created with <a
  href="https://github.com/workflowr/workflowr">workflowr</a> (version
1.7.0). The <em>Checks</em> tab describes the reproducibility checks
that were applied when the results were created. The <em>Past
versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguptodate">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>R Markdown file:</strong> up-to-date
</a>
</p>
</div>
<div id="strongRMarkdownfilestronguptodate"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great! Since the R Markdown file has been committed to the Git
repository, you know the exact version of the code that produced these
results.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the
global environment can affect the analysis in your R Markdown file in
unknown ways. For reproduciblity it’s best to always run the code in an
empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed20221022code">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Seed:</strong>
<code>set.seed(20221022)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed20221022code"
class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(20221022)</code> was run prior to running
the code in the R Markdown file. Setting a seed ensures that any results
that rely on randomness, e.g. subsampling or permutations, are
reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Session information:</strong>
recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package
versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be
confident that you successfully produced the results during this
run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongFilepathsstrongabsolute">
<span class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span> <strong>File paths:</strong> absolute </a>
</p>
</div>
<div id="strongFilepathsstrongabsolute" class="panel-collapse collapse">
<div class="panel-body">
<p>
Using absolute paths to the files within your workflowr project makes it
difficult for you and others to run your code on a different machine.
Change the absolute path(s) below to the suggested relative path(s) to
make your code more reproducible.
</p>
<table class="table table-condensed table-hover">
<thead>
<tr>
<th style="text-align:left;">
absolute
</th>
<th style="text-align:left;">
relative
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
~/Desktop/myproject/data/mixed_impact_subset_2022/data_clinical_patient.txt
</td>
<td style="text-align:left;">
data/mixed_impact_subset_2022/data_clinical_patient.txt
</td>
</tr>
<tr>
<td style="text-align:left;">
~/Desktop/myproject/data/mixed_impact_subset_2022/data_clinical_sample.txt
</td>
<td style="text-align:left;">
data/mixed_impact_subset_2022/data_clinical_sample.txt
</td>
</tr>
<tr>
<td style="text-align:left;">
~/Desktop/myproject/data/mixed_impact_subset_2022/data_cna.txt
</td>
<td style="text-align:left;">
data/mixed_impact_subset_2022/data_cna.txt
</td>
</tr>
<tr>
<td style="text-align:left;">
~/Desktop/myproject/data/mixed_impact_subset_2022/data_mutations.txt
</td>
<td style="text-align:left;">
data/mixed_impact_subset_2022/data_mutations.txt
</td>
</tr>
<tr>
<td style="text-align:left;">
~/Desktop/myproject/data/mixed_impact_subset_2022_clinical_data.txt
</td>
<td style="text-align:left;">
data/mixed_impact_subset_2022_clinical_data.txt
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrongahrefhttpsgithubcomAkhileshpandey87myprojecttreea556d9303b84391aa7f628194d81eb065c4a91cetargetblanka556d93a">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Repository version:</strong>
<a href="https://github.com/Akhileshpandey87/myproject/tree/a556d9303b84391aa7f628194d81eb065c4a91ce" target="_blank">a556d93</a>
</a>
</p>
</div>
<div
id="strongRepositoryversionstrongahrefhttpsgithubcomAkhileshpandey87myprojecttreea556d9303b84391aa7f628194d81eb065c4a91cetargetblanka556d93a"
class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development
and connecting the code version to the results is critical for
reproducibility.
</p>
<p>
The results in this page were generated with repository version
<a href="https://github.com/Akhileshpandey87/myproject/tree/a556d9303b84391aa7f628194d81eb065c4a91ce" target="_blank">a556d93</a>.
See the <em>Past versions</em> tab to see a history of the changes made
to the R Markdown and HTML files.
</p>
<p>
Note that you need to be careful to ensure that all relevant files for
the analysis have been committed to Git prior to generating the results
(you can use <code>wflow_publish</code> or
<code>wflow_git_commit</code>). workflowr only checks the R Markdown
file, but you know if there are other scripts or data files that it
depends on. Below is the status of the Git repository when the results
were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .Rhistory
    Ignored:    .Rproj.user/
    Ignored:    R-output_files/

Untracked files:
    Untracked:  data/mixed_impact_subset_2022/
    Untracked:  data/mixed_impact_subset_2022_clinical_data.txt
    Untracked:  data/pancan_pcawg_2020/
    Untracked:  data/pancan_pcawg_2020_clinical_data.txt
    Untracked:  site_libs/

Unstaged changes:
    Modified:   analysis/R-output.Rmd

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not
included in this status report because it is ok for generated content to
have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">

<p>
These are the previous versions of the repository in which changes were
made to the R Markdown (<code>analysis/msk_impact_result.Rmd</code>) and
HTML (<code>docs/msk_impact_result.html</code>) files. If you’ve
configured a remote Git repository (see <code>?wflow_git_remote</code>),
click on the hyperlinks in the table below to view the files as they
were in that past version.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/Akhileshpandey87/myproject/blob/a556d9303b84391aa7f628194d81eb065c4a91ce/analysis/msk_impact_result.Rmd" target="_blank">a556d93</a>
</td>
<td>
Akhilesh
</td>
<td>
2022-11-17
</td>
<td>
wflow_publish("analysis/msk_impact_result.Rmd")
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/Akhileshpandey87/myproject/59a382bfd5234b47b7f2a95955b3b5cdbc1b1d67/docs/msk_impact_result.html" target="_blank">59a382b</a>
</td>
<td>
Akhilesh
</td>
<td>
2022-11-14
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/Akhileshpandey87/myproject/blob/576f7cf57eec0a0e77972d148fbb66b0c636d216/analysis/msk_impact_result.Rmd" target="_blank">576f7cf</a>
</td>
<td>
Akhilesh
</td>
<td>
2022-11-14
</td>
<td>
wflow_publish("analysis/msk_impact_result.Rmd")
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/Akhileshpandey87/myproject/ba67e472d28ca6c3677b15320c1bf129b632d636/docs/msk_impact_result.html" target="_blank">ba67e47</a>
</td>
<td>
Akhilesh
</td>
<td>
2022-11-14
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/Akhileshpandey87/myproject/blob/ba427351f63aaa0a4e2b85a80123af6321b2522f/analysis/msk_impact_result.Rmd" target="_blank">ba42735</a>
</td>
<td>
Akhilesh
</td>
<td>
2022-11-14
</td>
<td>
wflow_publish("analysis/msk_impact_result.Rmd")
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<p>#Pan-Cancer MSK-IMPACT MET Validation Cohort (MSK 2022)</p>
<pre class="r"><code>mskdat_p &lt;- read.delim(&quot;~/Desktop/myproject/data/mixed_impact_subset_2022/data_clinical_patient.txt&quot;,header=TRUE,stringsAsFactor = TRUE,comment.char = &quot;#&quot;)
  head(mskdat_p)</code></pre>
<pre><code>  PATIENT_ID CRDB_ADJ_TXT CRDB_NOSYSTXT CRDB_PRIOR_RX CRDB_BRAINMET CRDB_ECOG
1  P-0000112                         NA                                    NA
2  P-0000113                         NA                                    NA
3  P-0000133                         NA                                    NA
4  P-0000190                         NA                                    NA
5  P-0000208                         NA                                    NA
6  P-0000219                         NA                                    NA
  OTHER_PATIENT_ID PARTA_CONSENTED_12_245 PARTC_CONSENTED_12_245 AGE_CURRENT
1          DMP0096                     NO                     NO          NA
2          DMP0101                    YES                     NO          68
3          DMP0116                    YES                     NO          83
4          DMP0178                    YES                     NO          50
5          DMP0193                    YES                     NO          69
6          DMP0192                    YES                     NO          63
   RACE RELIGION    SEX                 ETHNICITY OS_STATUS OS_MONTHS PED_IND
1                                                                  NA        
2 WHITE   JEWISH Female Non-Spanish; Non-Hispanic    LIVING   100.570      No
3 WHITE   JEWISH Female Non-Spanish; Non-Hispanic  DECEASED    18.904      No
4 WHITE   JEWISH Female Non-Spanish; Non-Hispanic  DECEASED    14.367      No
5 WHITE   JEWISH Female Non-Spanish; Non-Hispanic  DECEASED    12.855      No
6 WHITE   JEWISH Female Non-Spanish; Non-Hispanic    LIVING    99.518      No
  AGE SEX_DMT OS_STATUS_DMT OS_MONTHS_DMT DFS_STATUS DFS_MONTHS
1  NA                                  NA                    NA
2  NA                                  NA                    NA
3  NA                                  NA                    NA
4  NA                                  NA                    NA
5  NA                                  NA                    NA
6  NA                                  NA                    NA
  HISTOLOGICAL_DIAGNOSIS CRDB_CONSENT_DATE_DAYS CRDB_SURVIVAL_STATUS
1                                            NA                     
2                                         21788                Alive
3                                         29793                 Dead
4                                         17840                 Dead
5                                         24948                 Dead
6                                         20058                Alive
  CRDB_TREATMENT_END_DAYS CRDB_OFF_STUDY_DAYS CRDB_BASIC_COMMENTS
1                      NA                  NA                  NA
2                      NA                  NA                  NA
3                   30399                  NA                  NA
4                   18314                  NA                  NA
5                   25378               25378                  NA
6                      NA                  NA                  NA</code></pre>
<pre class="r"><code>mskdat_s &lt;- read.delim (&quot;~/Desktop/myproject/data/mixed_impact_subset_2022/data_clinical_sample.txt&quot;,header=TRUE,stringsAsFactor = TRUE,comment.char = &quot;#&quot;)
head(mskdat_s)</code></pre>
<pre><code>          SAMPLE_ID PATIENT_ID DATE_ADDED MONTH_ADDED   WEEK_ADDED
1 P-0000112-T01-IM3  P-0000112 2015/04/07     2015/04 2015, Wk. 15
2 P-0000113-T01-IM3  P-0000113 2015/04/07     2015/04 2015, Wk. 15
3 P-0000133-T01-IM3  P-0000133 2015/04/07     2015/04 2015, Wk. 15
4 P-0000190-T01-IM3  P-0000190 2015/04/07     2015/04 2015, Wk. 15
5 P-0000208-T01-IM3  P-0000208 2015/04/07     2015/04 2015, Wk. 15
6 P-0000219-T01-IM3  P-0000219 2015/04/07     2015/04 2015, Wk. 15
  CRDB_SURVEY_COMMENTS                CANCER_TYPE SAMPLE_TYPE SAMPLE_CLASS
1                   NA     Esophagogastric Cancer     Primary        Tumor
2                   NA     Small Cell Lung Cancer     Primary        Tumor
3                   NA Non-Small Cell Lung Cancer     Primary        Tumor
4                   NA     Small Cell Lung Cancer  Metastasis        Tumor
5                   NA Non-Small Cell Lung Cancer     Primary        Tumor
6                   NA Non-Small Cell Lung Cancer     Primary        Tumor
  METASTATIC_SITE PRIMARY_SITE      CANCER_TYPE_DETAILED GENE_PANEL SO_COMMENTS
1  Not Applicable    Esophagus Esophageal Adenocarcinoma  IMPACT341            
2  Not Applicable         Lung    Small Cell Lung Cancer  IMPACT341            
3  Not Applicable         Lung       Lung Adenocarcinoma  IMPACT341            
4           Brain         Lung    Small Cell Lung Cancer  IMPACT341            
5  Not Applicable         Lung       Lung Adenocarcinoma  IMPACT341            
6  Not Applicable         Lung       Lung Adenocarcinoma  IMPACT341            
  SAMPLE_COVERAGE TUMOR_PURITY ONCOTREE_CODE
1             743           NA          ESCA
2             635           80          SCLC
3             529           30          LUAD
4            1216           90          SCLC
5            1346           70          LUAD
6            1033           70          LUAD
                                       MSI_COMMENT MSI_SCORE MSI_TYPE INSTITUTE
1 MICROSATELLITE STABLE (MSS). See MSI note below.      2.76   Stable     MSKCC
2                                    Not Available      1.08   Stable     MSKCC
3                                    Not Available      0.00   Stable     MSKCC
4                                    Not Available      2.02   Stable     MSKCC
5                                    Not Available      0.00   Stable     MSKCC
6                                    Not Available      0.08   Stable     MSKCC
  SOMATIC_STATUS AGE_AT_SEQ_REPORTED_YEARS ARCHER CVR_TMB_COHORT_PERCENTILE
1        Matched                        NA     NO                      84.5
2        Matched                        60     NO                      94.2
3        Matched                        82     NO                      58.6
4        Matched                        49     NO                      85.0
5        Matched                        69     NO                      29.9
6        Matched                        55     NO                      58.6
  CVR_TMB_SCORE CVR_TMB_TT_COHORT_PERCENTILE PATH_SLIDE_EXISTS MSK_SLIDE_ID
1           8.9                         86.1               YES       739966
2          20.1                         96.8                NO           NA
3           4.5                         42.2               YES       466714
4           8.9                         57.9                NO           NA
5           2.2                         19.9                NO           NA
6           4.5                         42.2               YES       468429
  WHO_GRADE MGMT_STATUS TMB_NONSYNONYMOUS
1                                8.873242
2                               19.964795
3                                4.436621
4                                8.873242
5                                2.218311
6                                4.436621</code></pre>
<pre class="r"><code>mskdat_cna &lt;- read.delim (&quot;~/Desktop/myproject/data/mixed_impact_subset_2022/data_cna.txt&quot;,header=TRUE,stringsAsFactor = TRUE,comment.char = &quot;#&quot;)
head(mskdat_cna)</code></pre>
<pre><code>  Hugo_Symbol P.0006972.T01.IM5 P.0035158.T01.IM6 P.0018646.T02.IM6
1        TAP1                 0                 0                 0
2      ERRFI1                 0                 0                 0
3       STK19                 0                 0                 0
4        CRKL                 0                 0                 0
5        SCG5                 0                 0                 0
6       STK11                 0                 0                 0
  P.0018311.T01.IM6 P.0022763.T01.IM6 P.0022198.T01.IM6 P.0036034.T01.IM6
1                 0                 0                 0                 0
2                 0                 0                 0                 0
3                 0                 0                 0                 0
4                 0                 0                 0                 0
5                 0                 0                 0                 0
6                 0                 0                 0                 0
  P.0001037.T01.IM3 P.0001833.T01.IM3 P.0029892.T04.IM6 P.0031459.T01.IM6
1                 0                 0                 0                 0
2                 0                 0                 0                 0
3                 0                 0                 0                 0
4                 0                 0                 0                 0
5                 0                 0                 0                 0
6                 0                 0                 0                 0
  P.0040381.T01.IM6 P.0005437.T01.IM5 P.0024180.T01.IM6 P.0001089.T01.IM3
1                 0                 0                 0                 0
2                 0                 0                 0                 0
3                 0                 0                 0                 0
4                 0                 0                 0                 0
5                 0                 0                 0                 0
6                 0                 0                 0                 0
  P.0031827.T03.IM6 P.0003424.T01.IM5 P.0028139.T02.IM6 P.0000112.T01.IM3
1                 0                 0                 0                 0
2                 0                 0                 0                 0
3                 0                 0                 0                 0
4                 0                 0                 0                 0
5                 0                 0                 0                 0
6                 0                 0                 0                 0
  P.0000616.T01.IM3 P.0031911.T02.IM6 P.0031970.T01.IM6 P.0018031.T01.IM6
1                 0                 0                 0                 0
2                 0                 0                 0                 0
3                 0                 0                 0                 0
4                 0                 0                 0                 0
5                 0                 0                 0                 0
6                 0                 0                 0                 0
  P.0017974.T01.IM6 P.0000219.T01.IM3 P.0026365.T01.IM6 P.0000339.T01.IM3
1                 0                 0                 0                 0
2                 0                 0                 0                 0
3                 0                 0                 0                 0
4                 0                 0                 0                 0
5                 0                 0                 0                 0
6                 0                 0                 0                 0
  P.0030643.T01.IM6 P.0025100.T02.IM6 P.0000208.T01.IM3 P.0002401.T01.IM3
1                 0                 0                 0                 0
2                 0                 0                 0                 0
3                 0                 0                 0                 0
4                 0                 0                 0                 0
5                 0                 0                 0                 0
6                 0                 0                 0                 0
  P.0027892.T01.IM6 P.0005027.T01.IM5 P.0022809.T01.IM6 P.0035786.T01.IM6
1                 0                 0                 0                 0
2                 0                 0                 0                 0
3                 0                 0                 0                 0
4                 0                 0                 0                 0
5                 0                 0                 0                 0
6                 0                 0                 0                 0
  P.0000807.T01.IM3 P.0000133.T01.IM3 P.0016885.T01.IM6 P.0036249.T04.IM6
1                 0                 0                 0                 0
2                 0                 0                 0                 0
3                 0                 0                 0                 0
4                 0                 0                 0                 0
5                 0                 0                 0                 0
6                 0                 0                 0                 0
  P.0001790.T01.IM3 P.0004044.T01.IM3 P.0019852.T01.IM6 P.0023094.T01.IM6
1                 0                 0                 0                 0
2                 0                 0                 0                 0
3                 0                 0                 0                 0
4                 0                 0                 0                 2
5                 0                 0                 0                 0
6                 0                 0                 0                 0
  P.0026121.T01.IM6 P.0009423.T02.IM6 P.0044127.T01.IM6 P.0015075.T01.IM6
1                 0                 0                 0                 0
2                 0                 0                 0                 0
3                 0                 0                 0                 0
4                 0                 0                 0                 0
5                 0                 0                 0                 0
6                 0                 0                 0                 0
  P.0017479.T02.IM6 P.0012160.T03.IM6 P.0030727.T01.IM6 P.0000190.T01.IM3
1                 0                 0                 0                 0
2                 0                 0                 0                 0
3                 0                 0                 0                 0
4                 0                 0                 0                 0
5                 0                 0                 0                 0
6                 0                 0                 0                 0
  P.0035774.T01.IM6 P.0003935.T01.IM3 P.0034579.T01.IM6 P.0000282.T02.IM5
1                 0                 0                 0                 0
2                 0                 0                 0                 0
3                 0                 0                 0                 0
4                 0                 0                 0                 0
5                 0                 0                 0                 0
6                 0                 0                 0                 0
  P.0027762.T01.IM6 P.0006750.T01.IM5 P.0036267.T01.IM6 P.0003964.T01.IM3
1                 0                 0                 0                 0
2                 0                 0                 0                 0
3                 0                 0                 0                 0
4                 0                 0                 0                 0
5                 0                 0                 0                 0
6                 0                 0                 0                 0
  P.0035162.T01.IM6 P.0003030.T01.IM3 P.0019217.T01.IM6 P.0003836.T01.IM5
1                 0                 0                 0                 0
2                 0                 0                 2                 0
3                 0                 0                 0                 0
4                 0                 0                 0                 0
5                 0                 0                 0                 0
6                 0                 0                 0                 0
  P.0000113.T01.IM3 P.0031751.T01.IM6 P.0003250.T01.IM5 P.0018267.T01.IM6
1                 0                 0                 0                 0
2                 0                 0                 0                 0
3                 0                 0                 0                 0
4                 0                 0                 0                 0
5                 0                 0                 0                 0
6                 0                 0                 0                 0
  P.0002000.T01.IM3 P.0009789.T02.IM6
1                 0                 0
2                 0                 0
3                 0                 0
4                 0                 0
5                 0                 0
6                 0                 0</code></pre>
<pre class="r"><code>mskdat_m &lt;- read.delim (&quot;~/Desktop/myproject/data/mixed_impact_subset_2022/data_mutations.txt&quot;,header=TRUE,stringsAsFactor = TRUE,comment.char = &quot;#&quot;)
head(mskdat_m)</code></pre>
<pre><code>  Hugo_Symbol Entrez_Gene_Id Center NCBI_Build Chromosome Start_Position
1         SYK           6850  MSKCC     GRCh37          9       93606487
2        TP53           7157  MSKCC     GRCh37         17        7577121
3         RB1           5925  MSKCC     GRCh37         13       48936995
4       HLA-A           3105  MSKCC     GRCh37          6       29910371
5         NF1           4763  MSKCC     GRCh37         17       29556370
6         MET           4233  MSKCC     GRCh37          7      116411887
  End_Position Strand        Consequence Variant_Classification Variant_Type
1     93606487      +   missense_variant      Missense_Mutation          SNP
2      7577121      +   missense_variant      Missense_Mutation          SNP
3     48936995      +        stop_gained      Nonsense_Mutation          SNP
4     29910372      + frameshift_variant        Frame_Shift_Ins          INS
5     29556370      +   missense_variant      Missense_Mutation          SNP
6    116411888      +     intron_variant                 Intron          INS
  Reference_Allele Tumor_Seq_Allele1 Tumor_Seq_Allele2    dbSNP_RS
1                C                 C                 T            
2                G                 G                 A rs121913343
3                C                 C                 T rs587778842
4                -                 -                 G            
5                A                 A                 G            
6                -                 -                AG            
  dbSNP_Val_Status Tumor_Sample_Barcode Matched_Norm_Sample_Barcode
1               NA    P-0023094-T01-IM6                          NA
2               NA    P-0035774-T01-IM6                          NA
3               NA    P-0035774-T01-IM6                          NA
4               NA    P-0035774-T01-IM6                          NA
5               NA    P-0035774-T01-IM6                          NA
6               NA    P-0035774-T01-IM6                          NA
  Match_Norm_Seq_Allele1 Match_Norm_Seq_Allele2 Tumor_Validation_Allele1
1                     NA                     NA                       NA
2                     NA                     NA                       NA
3                     NA                     NA                       NA
4                     NA                     NA                       NA
5                     NA                     NA                       NA
6                     NA                     NA                       NA
  Tumor_Validation_Allele2 Match_Norm_Validation_Allele1
1                       NA                            NA
2                       NA                            NA
3                       NA                            NA
4                       NA                            NA
5                       NA                            NA
6                       NA                            NA
  Match_Norm_Validation_Allele2 Verification_Status Validation_Status
1                            NA                  NA           Unknown
2                            NA                  NA           Unknown
3                            NA                  NA           Unknown
4                            NA                  NA           Unknown
5                            NA                  NA           Unknown
6                            NA                  NA           Unknown
  Mutation_Status Sequencing_Phase Sequence_Source Validation_Method      Score
1         SOMATIC               NA              NA                NA MSK-IMPACT
2         SOMATIC               NA              NA                NA MSK-IMPACT
3         SOMATIC               NA              NA                NA MSK-IMPACT
4         SOMATIC               NA              NA                NA MSK-IMPACT
5         SOMATIC               NA              NA                NA MSK-IMPACT
6         SOMATIC               NA              NA                NA MSK-IMPACT
  BAM_File Sequencer t_ref_count t_alt_count n_ref_count n_alt_count
1       NA        NA         640         230         898           0
2       NA        NA         379         240         769           0
3       NA        NA         231         134         426           0
4       NA        NA         595          88         800           0
5       NA        NA         525         147         654           0
6       NA        NA         514        1972        1087           0
                                     HGVSc             HGVSp HGVSp_Short
1               ENST00000375746.1:c.307C&gt;T       p.Leu103Phe     p.L103F
2               ENST00000269305.4:c.817C&gt;T       p.Arg273Cys     p.R273C
3               ENST00000267163.4:c.763C&gt;T       p.Arg255Ter     p.R255*
4                ENST00000376809.5:c.46dup p.Ala16GlyfsTer83 p.A16Gfs*83
5              ENST00000358273.4:c.2737A&gt;G       p.Ile913Val     p.I913V
6 ENST00000397752.3:c.2888-16_2888-15insAG                       p.*963*
    Transcript_ID         RefSeq Protein_position   Codons Exon_Number COMMENTS
1 ENST00000375746 NM_001174167.1              103  Ctc/Ttc          NA       NA
2 ENST00000269305 NM_001126112.2              273  Cgt/Tgt          NA       NA
3 ENST00000267163    NM_000321.2              255  Cga/Tga          NA       NA
4 ENST00000376809    NM_002116.7               14 tcg/tcGg          NA       NA
5 ENST00000358273 NM_001042492.2              913  Ata/Gta          NA       NA
6 ENST00000397752    NM_000245.2               NA                   NA       NA
  AA_MAF AFR_MAF ALLELE_NUM AMR_MAF ASN_MAF Allele Amino_Acid_Change
1     NA      NA                 NA      NA     NA                NA
2     NA      NA                 NA      NA     NA                NA
3     NA      NA                 NA      NA     NA                NA
4     NA      NA                 NA      NA     NA                NA
5     NA      NA                 NA      NA     NA                NA
6     NA      NA                 NA      NA     NA                NA
  Amino_acids BIOTYPE CANONICAL CCDS CDS_position CLIN_SIG Comments DISTANCE
1          NA      NA        NA   NA           NA       NA       NA       NA
2          NA      NA        NA   NA           NA       NA       NA       NA
3          NA      NA        NA   NA           NA       NA       NA       NA
4          NA      NA        NA   NA           NA       NA       NA       NA
5          NA      NA        NA   NA           NA       NA       NA       NA
6          NA      NA        NA   NA           NA       NA       NA       NA
  DOMAINS EAS_MAF EA_MAF ENSP EUR_MAF EXON ExAC_AF ExAC_AF_AFR ExAC_AF_AMR
1      NA      NA     NA   NA      NA   NA      NA          NA          NA
2      NA      NA     NA   NA      NA   NA      NA          NA          NA
3      NA      NA     NA   NA      NA   NA      NA          NA          NA
4      NA      NA     NA   NA      NA   NA      NA          NA          NA
5      NA      NA     NA   NA      NA   NA      NA          NA          NA
6      NA      NA     NA   NA      NA   NA      NA          NA          NA
  ExAC_AF_EAS ExAC_AF_FIN ExAC_AF_NFE ExAC_AF_OTH ExAC_AF_SAS
1          NA          NA          NA          NA          NA
2          NA          NA          NA          NA          NA
3          NA          NA          NA          NA          NA
4          NA          NA          NA          NA          NA
5          NA          NA          NA          NA          NA
6          NA          NA          NA          NA          NA
  Existing_variation FILTER Feature Feature_type GENE_PHENO GMAF Gene HGNC_ID
1                 NA     NA      NA           NA         NA   NA   NA      NA
2                 NA     NA      NA           NA         NA   NA   NA      NA
3                 NA     NA      NA           NA         NA   NA   NA      NA
4                 NA     NA      NA           NA         NA   NA   NA      NA
5                 NA     NA      NA           NA         NA   NA   NA      NA
6                 NA     NA      NA           NA         NA   NA   NA      NA
  HGVS_OFFSET HIGH_INF_POS IMPACT INTRON IS_NEW MA.FIS MA.FImpact MA.link.MSA
1          NA           NA     NA     NA            NA         NA          NA
2          NA           NA     NA     NA            NA         NA          NA
3          NA           NA     NA     NA            NA         NA          NA
4          NA           NA     NA     NA            NA         NA          NA
5          NA           NA     NA     NA            NA         NA          NA
6          NA           NA     NA     NA            NA         NA          NA
  MA.link.PDB MA.link.var MA.protein.change MINIMISED MOTIF_NAME MOTIF_POS
1          NA          NA                NA        NA         NA        NA
2          NA          NA                NA        NA         NA        NA
3          NA          NA                NA        NA         NA        NA
4          NA          NA                NA        NA         NA        NA
5          NA          NA                NA        NA         NA        NA
6          NA          NA                NA        NA         NA        NA
  MOTIF_SCORE_CHANGE PHENO PICK PUBMED PolyPhen SAS_MAF SIFT SOMATIC SWISSPROT
1                 NA    NA   NA     NA       NA      NA   NA      NA        NA
2                 NA    NA   NA     NA       NA      NA   NA      NA        NA
3                 NA    NA   NA     NA       NA      NA   NA      NA        NA
4                 NA    NA   NA     NA       NA      NA   NA      NA        NA
5                 NA    NA   NA     NA       NA      NA   NA      NA        NA
6                 NA    NA   NA     NA       NA      NA   NA      NA        NA
  SYMBOL SYMBOL_SOURCE TREMBL TSL Transcript UNIPARC VARIANT_CLASS all_effects
1     NA            NA     NA  NA         NA      NA            NA          NA
2     NA            NA     NA  NA         NA      NA            NA          NA
3     NA            NA     NA  NA         NA      NA            NA          NA
4     NA            NA     NA  NA         NA      NA            NA          NA
5     NA            NA     NA  NA         NA      NA            NA          NA
6     NA            NA     NA  NA         NA      NA            NA          NA
  amino_acid_change cDNA_Change cDNA_position cdna_change comments n_depth
1                NA          NA            NA          NA       NA      NA
2                NA          NA            NA          NA       NA      NA
3                NA          NA            NA          NA       NA      NA
4                NA          NA            NA          NA       NA      NA
5                NA          NA            NA          NA       NA      NA
6                NA          NA            NA          NA       NA      NA
  t_depth transcript
1      NA         NA
2      NA         NA
3      NA         NA
4      NA         NA
5      NA         NA
6      NA         NA</code></pre>
<pre class="r"><code>mskdat_cd &lt;- read.delim (&quot;~/Desktop/myproject/data/mixed_impact_subset_2022_clinical_data.txt&quot;,header=TRUE,stringsAsFactor = TRUE,comment.char = &quot;#&quot;)
head(mskdat_cd)</code></pre>
<pre><code>                  Study.ID Patient.ID         Sample.ID Diagnosis.Age
1 mixed_impact_subset_2022  P-0000112 P-0000112-T01-IM3            NA
2 mixed_impact_subset_2022  P-0000113 P-0000113-T01-IM3            NA
3 mixed_impact_subset_2022  P-0000133 P-0000133-T01-IM3            NA
4 mixed_impact_subset_2022  P-0000190 P-0000190-T01-IM3            NA
5 mixed_impact_subset_2022  P-0000208 P-0000208-T01-IM3            NA
6 mixed_impact_subset_2022  P-0000219 P-0000219-T01-IM3            NA
  Age.at.Which.Sequencing.was.Reported..Years. Patient.Current.Age Archer.Panel
1                                           NA                  NA           NO
2                                           60                  68           NO
3                                           82                  83           NO
4                                           49                  50           NO
5                                           69                  69           NO
6                                           55                  63           NO
                 Cancer.Type      Cancer.Type.Detailed CRDB_ADJ_TXT
1     Esophagogastric Cancer Esophageal Adenocarcinoma         &lt;NA&gt;
2     Small Cell Lung Cancer    Small Cell Lung Cancer         &lt;NA&gt;
3 Non-Small Cell Lung Cancer       Lung Adenocarcinoma         &lt;NA&gt;
4     Small Cell Lung Cancer    Small Cell Lung Cancer         &lt;NA&gt;
5 Non-Small Cell Lung Cancer       Lung Adenocarcinoma         &lt;NA&gt;
6 Non-Small Cell Lung Cancer       Lung Adenocarcinoma         &lt;NA&gt;
  CRDB_BASIC_COMMENTS CRDB_BRAINMET CRDB_CONSENT_DATE_DAYS CRDB_ECOG
1                  NA          &lt;NA&gt;                     NA        NA
2                  NA          &lt;NA&gt;                  21788        NA
3                  NA          &lt;NA&gt;                  29793        NA
4                  NA          &lt;NA&gt;                  17840        NA
5                  NA          &lt;NA&gt;                  24948        NA
6                  NA          &lt;NA&gt;                  20058        NA
  CRDB_NOSYSTXT CRDB_OFF_STUDY_DAYS CRDB_PRIOR_RX CRDB_SURVEY_COMMENTS
1            NA                  NA          &lt;NA&gt;                   NA
2            NA                  NA          &lt;NA&gt;                   NA
3            NA                  NA          &lt;NA&gt;                   NA
4            NA                  NA          &lt;NA&gt;                   NA
5            NA               25378          &lt;NA&gt;                   NA
6            NA                  NA          &lt;NA&gt;                   NA
  CRDB_SURVIVAL_STATUS CRDB_TREATMENT_END_DAYS
1                 &lt;NA&gt;                      NA
2                Alive                      NA
3                 Dead                   30399
4                 Dead                   18314
5                 Dead                   25378
6                Alive                      NA
  Impact.TMB.Percentile..Across.All.Tumor.Types. Impact.TMB.Score
1                                           84.5              8.9
2                                           94.2             20.1
3                                           58.6              4.5
4                                           85.0              8.9
5                                           29.9              2.2
6                                           58.6              4.5
  Impact.TMB.Percentile..Within.Tumor.Type. Date.added.to.cBioPortal
1                                      86.1               2015/04/07
2                                      96.8               2015/04/07
3                                      42.2               2015/04/07
4                                      57.9               2015/04/07
5                                      19.9               2015/04/07
6                                      42.2               2015/04/07
  Disease.Free..Months. Disease.Free.Status        Ethnicity.Category
1                    NA                &lt;NA&gt;                      &lt;NA&gt;
2                    NA                &lt;NA&gt; Non-Spanish; Non-Hispanic
3                    NA                &lt;NA&gt; Non-Spanish; Non-Hispanic
4                    NA                &lt;NA&gt; Non-Spanish; Non-Hispanic
5                    NA                &lt;NA&gt; Non-Spanish; Non-Hispanic
6                    NA                &lt;NA&gt; Non-Spanish; Non-Hispanic
  Fraction.Genome.Altered Gene.Panel Neoplasm.Histologic.Type.Name
1                  0.3580  IMPACT341                          &lt;NA&gt;
2                  0.3740  IMPACT341                          &lt;NA&gt;
3                  0.0026  IMPACT341                          &lt;NA&gt;
4                  0.4202  IMPACT341                          &lt;NA&gt;
5                  0.1173  IMPACT341                          &lt;NA&gt;
6                  0.0001  IMPACT341                          &lt;NA&gt;
  Institute.Source Metastatic.Site MGMT.Status Month.added.to.cBioPortal
1            MSKCC            &lt;NA&gt;        &lt;NA&gt;                   2015/04
2            MSKCC            &lt;NA&gt;        &lt;NA&gt;                   2015/04
3            MSKCC            &lt;NA&gt;        &lt;NA&gt;                   2015/04
4            MSKCC           Brain        &lt;NA&gt;                   2015/04
5            MSKCC            &lt;NA&gt;        &lt;NA&gt;                   2015/04
6            MSKCC            &lt;NA&gt;        &lt;NA&gt;                   2015/04
                                       MSI.Comment MSI.Score MSI.Type
1 MICROSATELLITE STABLE (MSS). See MSI note below.      2.76   Stable
2                                             &lt;NA&gt;      1.08   Stable
3                                             &lt;NA&gt;      0.00   Stable
4                                             &lt;NA&gt;      2.02   Stable
5                                             &lt;NA&gt;      0.00   Stable
6                                             &lt;NA&gt;      0.08   Stable
  MSK.Slide.ID Mutation.Count Oncotree.Code Overall.Survival..Months.
1       739966              8          ESCA                        NA
2           NA             18          SCLC                   100.570
3       466714              4          LUAD                    18.904
4           NA              8          SCLC                    14.367
5           NA              2          LUAD                    12.855
6       468429              4          LUAD                    99.518
  Overall.Survival.Months.Reported.by.DMT Overall.Survival.Status
1                                      NA                    &lt;NA&gt;
2                                      NA                0:LIVING
3                                      NA              1:DECEASED
4                                      NA              1:DECEASED
5                                      NA              1:DECEASED
6                                      NA                0:LIVING
  Overall.Status.Reported.by.DMT Other.Patient.ID X12.245.Part.A.Consented
1                           &lt;NA&gt;          DMP0096                       NO
2                           &lt;NA&gt;          DMP0101                      YES
3                           &lt;NA&gt;          DMP0116                      YES
4                           &lt;NA&gt;          DMP0178                      YES
5                           &lt;NA&gt;          DMP0193                      YES
6                           &lt;NA&gt;          DMP0192                      YES
  X12.245.Part.C.Consented MSK.Pathology.Slide.Available
1                       NO                           YES
2                       NO                            NO
3                       NO                           YES
4                       NO                            NO
5                       NO                            NO
6                       NO                           YES
  Pediatric.Case.Indicator Primary.Tumor.Site Race.Category Religion
1                     &lt;NA&gt;          Esophagus          &lt;NA&gt;     &lt;NA&gt;
2                       No               Lung         WHITE   JEWISH
3                       No               Lung         WHITE   JEWISH
4                       No               Lung         WHITE   JEWISH
5                       No               Lung         WHITE   JEWISH
6                       No               Lung         WHITE   JEWISH
  Sample.Class Number.of.Samples.Per.Patient Sample.coverage Sample.Type    Sex
1        Tumor                             1             743     Primary   &lt;NA&gt;
2        Tumor                             1             635     Primary Female
3        Tumor                             1             529     Primary Female
4        Tumor                             1            1216  Metastasis Female
5        Tumor                             1            1346     Primary Female
6        Tumor                             1            1033     Primary Female
  Sex.Reported.by.DMT Somatic.Status SO.comments TMB..nonsynonymous.
1                &lt;NA&gt;        Matched        &lt;NA&gt;            8.873242
2                &lt;NA&gt;        Matched        &lt;NA&gt;           19.964795
3                &lt;NA&gt;        Matched        &lt;NA&gt;            4.436621
4                &lt;NA&gt;        Matched        &lt;NA&gt;            8.873242
5                &lt;NA&gt;        Matched        &lt;NA&gt;            2.218311
6                &lt;NA&gt;        Matched        &lt;NA&gt;            4.436621
  Tumor.Purity Week.added.to.cBioPortal WHO.Grade
1           NA             2015, Wk. 15      &lt;NA&gt;
2           80             2015, Wk. 15      &lt;NA&gt;
3           30             2015, Wk. 15      &lt;NA&gt;
4           90             2015, Wk. 15      &lt;NA&gt;
5           70             2015, Wk. 15      &lt;NA&gt;
6           70             2015, Wk. 15      &lt;NA&gt;</code></pre>
<pre class="r"><code>#frequency Distribution of CANCER TYPE
library(scales)
table(mskdat_s$CANCER_TYPE)</code></pre>
<pre><code>
 Cancer of Unknown Primary          Colorectal Cancer 
                         1                          2 
        Endometrial Cancer     Esophagogastric Cancer 
                         3                          4 
           Germ Cell Tumor                     Glioma 
                         2                          5 
      Hepatobiliary Cancer                   Melanoma 
                         2                          2 
Non-Small Cell Lung Cancer             Ovarian Cancer 
                        36                          1 
         Pancreatic Cancer       Renal Cell Carcinoma 
                         1                          5 
    Small Cell Lung Cancer 
                         5 </code></pre>
<pre class="r"><code># generating frequency table
data &lt;- data.frame(table(mskdat_s$CANCER_TYPE))
# relative frequency
num_obsrv &lt;- sum(table(mskdat_s$CANCER_TYPE))
freq&lt;-data[,2]/num_obsrv
data_frame &lt;- data.frame(data, freq)
colnames(data_frame) &lt;- c(&quot;CANCER TYPE&quot;,&quot;#&quot;,&quot;freq&quot;)
# order in descending by frequency
data_frame&lt;-data_frame[order(data_frame$freq,decreasing = TRUE),]
#converting frequency in percentage with accuracy at 2 decimal place 
data_frame$freq &lt;- percent(data_frame$freq, accuracy=.01)
print (data_frame)</code></pre>
<pre><code>                  CANCER TYPE  #   freq
9  Non-Small Cell Lung Cancer 36 52.17%
6                      Glioma  5  7.25%
12       Renal Cell Carcinoma  5  7.25%
13     Small Cell Lung Cancer  5  7.25%
4      Esophagogastric Cancer  4  5.80%
3          Endometrial Cancer  3  4.35%
2           Colorectal Cancer  2  2.90%
5             Germ Cell Tumor  2  2.90%
7        Hepatobiliary Cancer  2  2.90%
8                    Melanoma  2  2.90%
1   Cancer of Unknown Primary  1  1.45%
10             Ovarian Cancer  1  1.45%
11          Pancreatic Cancer  1  1.45%</code></pre>
<pre class="r"><code>#frequency Distribution of CANCER TYPE DETAILED
library(scales)
table(mskdat_s$CANCER_TYPE_DETAILED)</code></pre>
<pre><code>
                                        Adenocarcinoma, NOS 
                                                          1 
                                       Colon Adenocarcinoma 
                                                          2 
                                         Cutaneous Melanoma 
                                                          2 
                                      Endometrial Carcinoma 
                                                          1 
                                  Esophageal Adenocarcinoma 
                                                          1 
                            Extrahepatic Cholangiocarcinoma 
                                                          1 
                                    Glioblastoma Multiforme 
                                                          3 
                                     High-Grade Glioma, NOS 
                                                          2 
                           High-Grade Serous Ovarian Cancer 
                                                          1 
                            Intrahepatic Cholangiocarcinoma 
                                                          1 
                                        Lung Adenocarcinoma 
                                                         31 
                                  Lung Neuroendocrine Tumor 
                                                          1 
                                      Mixed Germ Cell Tumor 
                                                          1 
                                 Non-Small Cell Lung Cancer 
                                                          2 
                                  Pancreatic Adenocarcinoma 
                                                          1 
                             Papillary Renal Cell Carcinoma 
                                                          1 
                          Pleomorphic Carcinoma of the Lung 
                                                          2 
                                       Renal Cell Carcinoma 
                                                          1 
                                 Renal Clear Cell Carcinoma 
                                                          3 
                                     Small Cell Lung Cancer 
                                                          5 
                                     Stomach Adenocarcinoma 
                                                          3 
Uterine Serous Carcinoma/Uterine Papillary Serous Carcinoma 
                                                          2 
                                             Yolk Sac Tumor 
                                                          1 </code></pre>
<pre class="r"><code># generating frequency table
data &lt;- data.frame(table(mskdat_s$CANCER_TYPE_DETAILED))
# relative frequency
num_obsrv &lt;- sum(table(mskdat_s$CANCER_TYPE_DETAILED))
freq&lt;-data[,2]/num_obsrv
data_frame &lt;- data.frame(data, freq)
colnames(data_frame) &lt;- c(&quot;CANCER TYPE DETAILED&quot;,&quot;#&quot;,&quot;freq&quot;)
# order in descending by frequency
data_frame&lt;-data_frame[order(data_frame$freq,decreasing = TRUE),]
#converting frequency in percentage with accuracy at 2 decimal place 
data_frame$freq &lt;- percent(data_frame$freq, accuracy=.01)
print (data_frame)</code></pre>
<pre><code>                                          CANCER TYPE DETAILED  #   freq
11                                         Lung Adenocarcinoma 31 44.93%
20                                      Small Cell Lung Cancer  5  7.25%
7                                      Glioblastoma Multiforme  3  4.35%
19                                  Renal Clear Cell Carcinoma  3  4.35%
21                                      Stomach Adenocarcinoma  3  4.35%
2                                         Colon Adenocarcinoma  2  2.90%
3                                           Cutaneous Melanoma  2  2.90%
8                                       High-Grade Glioma, NOS  2  2.90%
14                                  Non-Small Cell Lung Cancer  2  2.90%
17                           Pleomorphic Carcinoma of the Lung  2  2.90%
22 Uterine Serous Carcinoma/Uterine Papillary Serous Carcinoma  2  2.90%
1                                          Adenocarcinoma, NOS  1  1.45%
4                                        Endometrial Carcinoma  1  1.45%
5                                    Esophageal Adenocarcinoma  1  1.45%
6                              Extrahepatic Cholangiocarcinoma  1  1.45%
9                             High-Grade Serous Ovarian Cancer  1  1.45%
10                             Intrahepatic Cholangiocarcinoma  1  1.45%
12                                   Lung Neuroendocrine Tumor  1  1.45%
13                                       Mixed Germ Cell Tumor  1  1.45%
15                                   Pancreatic Adenocarcinoma  1  1.45%
16                              Papillary Renal Cell Carcinoma  1  1.45%
18                                        Renal Cell Carcinoma  1  1.45%
23                                              Yolk Sac Tumor  1  1.45%</code></pre>
<pre class="r"><code>#frequency Distribution of Mutated genes
library(scales)
table(mskdat_m$Hugo_Symbol)</code></pre>
<pre><code>
    ABL1  ANKRD11      APC       AR   ARID1A   ARID1B    ARID2    ASXL2 
       2        1        4        1        1        1        4        2 
     ATM      ATR     ATRX      AXL     BAP1     BCL2     BCOR      BLM 
       4        1        3        1        1        1        2        1 
    BRAF    BRCA1    BRCA2     BRD4    BRIP1      BTK     CALR   CARD11 
       4        3        3        1        1        1        1        2 
     CBL    CCND2    CD274    CDK12   CDKN1B    CENPA    CSDE1   CTNNB1 
       1        2        1        1        1        2        1        3 
    CUL3     CYLD     DDR2   DICER1    DOT1L     EGFR   EIF1AX     ELF3 
       1        1        2        2        4       20        1        1 
   EP300    EPAS1    EPHA3    EPHA5    EPHA7    EPHB1    ERBB2    ERBB4 
       3        1        1        4        3        1        2        4 
   ERCC3    ERCC4      ERG     ESR1     ETV6     FAT1    FBXW7    FGFR4 
       2        1        1        1        2        3        1        2 
  FIP1L1     FLT1     FLT3    FOXA1    FOXL2    FUBP1      FYN     GLI1 
       1        3        2        1        1        1        3        1 
  GRIN2A    GSK3B    H3F3A      HGF HIST1H1C    HLA-A   HOXB13     IDH1 
       4        1        1        5        1        1        1        3 
    IGF1    IGF1R    IKZF1     IL7R    INHBA   INPP4B     INSR     JAK2 
       1        1        1        2        2        2        1        2 
    JAK3    KDM5A    KDM5C      KDR    KEAP1      KIT     KLF4    KMT2B 
       1        1        4        2        5        2        2        2 
   KMT2C    KMT2D     KRAS    LATS1    LATS2     MDC1     MDM2      MET 
       2        6        6        1        1        1        1       11 
     MGA     MLH1      MLL     MLL3     MSH3    MST1R     MTOR      MYC 
       3        1        1        2        2        2        3        1 
    MYCN      NBN    NCOR1      NF1   NFE2L2   NOTCH1   NOTCH2   NOTCH3 
       1        3        2        3        1        1        1        3 
  NOTCH4     NPM1     NRAS     NSD1    NTRK1    NTRK2    NTRK3     PAK7 
       2        2        1        1        2        1        5        2 
    PAX5    PBRM1    PDCD1   PDGFRA   PDGFRB    PDPK1  PIK3C2G   PIK3C3 
       1        5        1        3        2        1        2        1 
  PIK3CA   PIK3CB   PIK3CD   PIK3CG   PIK3R1   PIK3R3     PMS2    POLD1 
       6        1        1        2        1        3        1        2 
    POLE    PPM1D  PPP2R1A    PREX2     PTEN   PTPN11    PTPRD    PTPRT 
       4        1        2        2        3        1        6        4 
    RAC2    RAD50    RASA1      RB1    RBM10   RECQL4      RET   RICTOR 
       1        2        1       10        1        1        3        1 
   RNF43     ROS1    RPTOR    RRAGC    RTEL1    SETD2    SF3B1     SLX4 
       3        3        1        1        1        6        2        1 
   SMAD4  SMARCA4      SMO     SOS1    SOX17     SOX2     SOX9     SPEN 
       2        3        1        2        1        1        1        1 
   STAG2   STAT5A    STK11    STK19      SYK     TAP1   TCF7L2      TEK 
       2        1        1        2        2        1        1        1 
    TERT     TET1     TET2     TOP1     TP53     TP63     TSC1     TSC2 
      10        2        1        1       41        2        1        1 
     VHL     XIAP    ZFHX3    ZRSR2 
       1        1        4        2 </code></pre>
<pre class="r"><code># generating frequency table
data &lt;- data.frame(table(mskdat_m$Hugo_Symbol))
# relative frequency
num_obsrv &lt;- 69
freq&lt;-data[,2]/num_obsrv
data_frame &lt;- data.frame(data, freq)
colnames(data_frame) &lt;- c(&quot;Mutated genes&quot;,&quot;#&quot;,&quot;freq&quot;)
# order in descending by frequency
data_frame&lt;-data_frame[order(data_frame$freq,decreasing = TRUE),]
#converting frequency in percentage with accuracy at 2 decimal place 
data_frame$freq &lt;- percent(data_frame$freq, accuracy=.01)
print (data_frame)</code></pre>
<pre><code>    Mutated genes  #   freq
181          TP53 41 59.42%
38           EGFR 20 28.99%
96            MET 11 15.94%
148           RB1 10 14.49%
177          TERT 10 14.49%
90          KMT2D  6  8.70%
91           KRAS  6  8.70%
129        PIK3CA  6  8.70%
143         PTPRD  6  8.70%
158         SETD2  6  8.70%
68            HGF  5  7.25%
85          KEAP1  5  7.25%
119         NTRK3  5  7.25%
122         PBRM1  5  7.25%
3             APC  4  5.80%
7           ARID2  4  5.80%
9             ATM  4  5.80%
17           BRAF  4  5.80%
37          DOT1L  4  5.80%
44          EPHA5  4  5.80%
48          ERBB4  4  5.80%
65         GRIN2A  4  5.80%
83          KDM5C  4  5.80%
137          POLE  4  5.80%
144         PTPRT  4  5.80%
187         ZFHX3  4  5.80%
11           ATRX  3  4.35%
18          BRCA1  3  4.35%
19          BRCA2  3  4.35%
32         CTNNB1  3  4.35%
41          EP300  3  4.35%
45          EPHA7  3  4.35%
54           FAT1  3  4.35%
58           FLT1  3  4.35%
63            FYN  3  4.35%
72           IDH1  3  4.35%
97            MGA  3  4.35%
103          MTOR  3  4.35%
106           NBN  3  4.35%
108           NF1  3  4.35%
112        NOTCH3  3  4.35%
124        PDGFRA  3  4.35%
134        PIK3R3  3  4.35%
141          PTEN  3  4.35%
151           RET  3  4.35%
153         RNF43  3  4.35%
154          ROS1  3  4.35%
162       SMARCA4  3  4.35%
1            ABL1  2  2.90%
8           ASXL2  2  2.90%
15           BCOR  2  2.90%
24         CARD11  2  2.90%
26          CCND2  2  2.90%
30          CENPA  2  2.90%
35           DDR2  2  2.90%
36         DICER1  2  2.90%
47          ERBB2  2  2.90%
49          ERCC3  2  2.90%
53           ETV6  2  2.90%
56          FGFR4  2  2.90%
59           FLT3  2  2.90%
76           IL7R  2  2.90%
77          INHBA  2  2.90%
78         INPP4B  2  2.90%
80           JAK2  2  2.90%
84            KDR  2  2.90%
86            KIT  2  2.90%
87           KLF4  2  2.90%
88          KMT2B  2  2.90%
89          KMT2C  2  2.90%
100          MLL3  2  2.90%
101          MSH3  2  2.90%
102         MST1R  2  2.90%
107         NCOR1  2  2.90%
113        NOTCH4  2  2.90%
114          NPM1  2  2.90%
117         NTRK1  2  2.90%
120          PAK7  2  2.90%
125        PDGFRB  2  2.90%
127       PIK3C2G  2  2.90%
132        PIK3CG  2  2.90%
136         POLD1  2  2.90%
139       PPP2R1A  2  2.90%
140         PREX2  2  2.90%
146         RAD50  2  2.90%
159         SF3B1  2  2.90%
161         SMAD4  2  2.90%
164          SOS1  2  2.90%
169         STAG2  2  2.90%
172         STK19  2  2.90%
173           SYK  2  2.90%
178          TET1  2  2.90%
182          TP63  2  2.90%
188         ZRSR2  2  2.90%
2         ANKRD11  1  1.45%
4              AR  1  1.45%
5          ARID1A  1  1.45%
6          ARID1B  1  1.45%
10            ATR  1  1.45%
12            AXL  1  1.45%
13           BAP1  1  1.45%
14           BCL2  1  1.45%
16            BLM  1  1.45%
20           BRD4  1  1.45%
21          BRIP1  1  1.45%
22            BTK  1  1.45%
23           CALR  1  1.45%
25            CBL  1  1.45%
27          CD274  1  1.45%
28          CDK12  1  1.45%
29         CDKN1B  1  1.45%
31          CSDE1  1  1.45%
33           CUL3  1  1.45%
34           CYLD  1  1.45%
39         EIF1AX  1  1.45%
40           ELF3  1  1.45%
42          EPAS1  1  1.45%
43          EPHA3  1  1.45%
46          EPHB1  1  1.45%
50          ERCC4  1  1.45%
51            ERG  1  1.45%
52           ESR1  1  1.45%
55          FBXW7  1  1.45%
57         FIP1L1  1  1.45%
60          FOXA1  1  1.45%
61          FOXL2  1  1.45%
62          FUBP1  1  1.45%
64           GLI1  1  1.45%
66          GSK3B  1  1.45%
67          H3F3A  1  1.45%
69       HIST1H1C  1  1.45%
70          HLA-A  1  1.45%
71         HOXB13  1  1.45%
73           IGF1  1  1.45%
74          IGF1R  1  1.45%
75          IKZF1  1  1.45%
79           INSR  1  1.45%
81           JAK3  1  1.45%
82          KDM5A  1  1.45%
92          LATS1  1  1.45%
93          LATS2  1  1.45%
94           MDC1  1  1.45%
95           MDM2  1  1.45%
98           MLH1  1  1.45%
99            MLL  1  1.45%
104           MYC  1  1.45%
105          MYCN  1  1.45%
109        NFE2L2  1  1.45%
110        NOTCH1  1  1.45%
111        NOTCH2  1  1.45%
115          NRAS  1  1.45%
116          NSD1  1  1.45%
118         NTRK2  1  1.45%
121          PAX5  1  1.45%
123         PDCD1  1  1.45%
126         PDPK1  1  1.45%
128        PIK3C3  1  1.45%
130        PIK3CB  1  1.45%
131        PIK3CD  1  1.45%
133        PIK3R1  1  1.45%
135          PMS2  1  1.45%
138         PPM1D  1  1.45%
142        PTPN11  1  1.45%
145          RAC2  1  1.45%
147         RASA1  1  1.45%
149         RBM10  1  1.45%
150        RECQL4  1  1.45%
152        RICTOR  1  1.45%
155         RPTOR  1  1.45%
156         RRAGC  1  1.45%
157         RTEL1  1  1.45%
160          SLX4  1  1.45%
163           SMO  1  1.45%
165         SOX17  1  1.45%
166          SOX2  1  1.45%
167          SOX9  1  1.45%
168          SPEN  1  1.45%
170        STAT5A  1  1.45%
171         STK11  1  1.45%
174          TAP1  1  1.45%
175        TCF7L2  1  1.45%
176           TEK  1  1.45%
179          TET2  1  1.45%
180          TOP1  1  1.45%
183          TSC1  1  1.45%
184          TSC2  1  1.45%
185           VHL  1  1.45%
186          XIAP  1  1.45%</code></pre>
<pre class="r"><code>#frequency Distribution of Mutated genes
library(scales)
table(mskdat_m$Hugo_Symbol)</code></pre>
<pre><code>
    ABL1  ANKRD11      APC       AR   ARID1A   ARID1B    ARID2    ASXL2 
       2        1        4        1        1        1        4        2 
     ATM      ATR     ATRX      AXL     BAP1     BCL2     BCOR      BLM 
       4        1        3        1        1        1        2        1 
    BRAF    BRCA1    BRCA2     BRD4    BRIP1      BTK     CALR   CARD11 
       4        3        3        1        1        1        1        2 
     CBL    CCND2    CD274    CDK12   CDKN1B    CENPA    CSDE1   CTNNB1 
       1        2        1        1        1        2        1        3 
    CUL3     CYLD     DDR2   DICER1    DOT1L     EGFR   EIF1AX     ELF3 
       1        1        2        2        4       20        1        1 
   EP300    EPAS1    EPHA3    EPHA5    EPHA7    EPHB1    ERBB2    ERBB4 
       3        1        1        4        3        1        2        4 
   ERCC3    ERCC4      ERG     ESR1     ETV6     FAT1    FBXW7    FGFR4 
       2        1        1        1        2        3        1        2 
  FIP1L1     FLT1     FLT3    FOXA1    FOXL2    FUBP1      FYN     GLI1 
       1        3        2        1        1        1        3        1 
  GRIN2A    GSK3B    H3F3A      HGF HIST1H1C    HLA-A   HOXB13     IDH1 
       4        1        1        5        1        1        1        3 
    IGF1    IGF1R    IKZF1     IL7R    INHBA   INPP4B     INSR     JAK2 
       1        1        1        2        2        2        1        2 
    JAK3    KDM5A    KDM5C      KDR    KEAP1      KIT     KLF4    KMT2B 
       1        1        4        2        5        2        2        2 
   KMT2C    KMT2D     KRAS    LATS1    LATS2     MDC1     MDM2      MET 
       2        6        6        1        1        1        1       11 
     MGA     MLH1      MLL     MLL3     MSH3    MST1R     MTOR      MYC 
       3        1        1        2        2        2        3        1 
    MYCN      NBN    NCOR1      NF1   NFE2L2   NOTCH1   NOTCH2   NOTCH3 
       1        3        2        3        1        1        1        3 
  NOTCH4     NPM1     NRAS     NSD1    NTRK1    NTRK2    NTRK3     PAK7 
       2        2        1        1        2        1        5        2 
    PAX5    PBRM1    PDCD1   PDGFRA   PDGFRB    PDPK1  PIK3C2G   PIK3C3 
       1        5        1        3        2        1        2        1 
  PIK3CA   PIK3CB   PIK3CD   PIK3CG   PIK3R1   PIK3R3     PMS2    POLD1 
       6        1        1        2        1        3        1        2 
    POLE    PPM1D  PPP2R1A    PREX2     PTEN   PTPN11    PTPRD    PTPRT 
       4        1        2        2        3        1        6        4 
    RAC2    RAD50    RASA1      RB1    RBM10   RECQL4      RET   RICTOR 
       1        2        1       10        1        1        3        1 
   RNF43     ROS1    RPTOR    RRAGC    RTEL1    SETD2    SF3B1     SLX4 
       3        3        1        1        1        6        2        1 
   SMAD4  SMARCA4      SMO     SOS1    SOX17     SOX2     SOX9     SPEN 
       2        3        1        2        1        1        1        1 
   STAG2   STAT5A    STK11    STK19      SYK     TAP1   TCF7L2      TEK 
       2        1        1        2        2        1        1        1 
    TERT     TET1     TET2     TOP1     TP53     TP63     TSC1     TSC2 
      10        2        1        1       41        2        1        1 
     VHL     XIAP    ZFHX3    ZRSR2 
       1        1        4        2 </code></pre>
<pre class="r"><code># generating frequency table
data &lt;- data.frame(table(mskdat_m$Hugo_Symbol))
# relative frequency
num_obsrv &lt;- 69
freq&lt;-data[,2]/num_obsrv
data_frame &lt;- data.frame(data, freq)
colnames(data_frame) &lt;- c(&quot;Mutated genes&quot;,&quot;#&quot;,&quot;freq&quot;)
# order in descending by frequency
data_frame&lt;-data_frame[order(data_frame$freq,decreasing = TRUE),]
#converting frequency in percentage with accuracy at 2 decimal place 
data_frame$freq &lt;- percent(data_frame$freq, accuracy=.01)
print (data_frame)</code></pre>
<pre><code>    Mutated genes  #   freq
181          TP53 41 59.42%
38           EGFR 20 28.99%
96            MET 11 15.94%
148           RB1 10 14.49%
177          TERT 10 14.49%
90          KMT2D  6  8.70%
91           KRAS  6  8.70%
129        PIK3CA  6  8.70%
143         PTPRD  6  8.70%
158         SETD2  6  8.70%
68            HGF  5  7.25%
85          KEAP1  5  7.25%
119         NTRK3  5  7.25%
122         PBRM1  5  7.25%
3             APC  4  5.80%
7           ARID2  4  5.80%
9             ATM  4  5.80%
17           BRAF  4  5.80%
37          DOT1L  4  5.80%
44          EPHA5  4  5.80%
48          ERBB4  4  5.80%
65         GRIN2A  4  5.80%
83          KDM5C  4  5.80%
137          POLE  4  5.80%
144         PTPRT  4  5.80%
187         ZFHX3  4  5.80%
11           ATRX  3  4.35%
18          BRCA1  3  4.35%
19          BRCA2  3  4.35%
32         CTNNB1  3  4.35%
41          EP300  3  4.35%
45          EPHA7  3  4.35%
54           FAT1  3  4.35%
58           FLT1  3  4.35%
63            FYN  3  4.35%
72           IDH1  3  4.35%
97            MGA  3  4.35%
103          MTOR  3  4.35%
106           NBN  3  4.35%
108           NF1  3  4.35%
112        NOTCH3  3  4.35%
124        PDGFRA  3  4.35%
134        PIK3R3  3  4.35%
141          PTEN  3  4.35%
151           RET  3  4.35%
153         RNF43  3  4.35%
154          ROS1  3  4.35%
162       SMARCA4  3  4.35%
1            ABL1  2  2.90%
8           ASXL2  2  2.90%
15           BCOR  2  2.90%
24         CARD11  2  2.90%
26          CCND2  2  2.90%
30          CENPA  2  2.90%
35           DDR2  2  2.90%
36         DICER1  2  2.90%
47          ERBB2  2  2.90%
49          ERCC3  2  2.90%
53           ETV6  2  2.90%
56          FGFR4  2  2.90%
59           FLT3  2  2.90%
76           IL7R  2  2.90%
77          INHBA  2  2.90%
78         INPP4B  2  2.90%
80           JAK2  2  2.90%
84            KDR  2  2.90%
86            KIT  2  2.90%
87           KLF4  2  2.90%
88          KMT2B  2  2.90%
89          KMT2C  2  2.90%
100          MLL3  2  2.90%
101          MSH3  2  2.90%
102         MST1R  2  2.90%
107         NCOR1  2  2.90%
113        NOTCH4  2  2.90%
114          NPM1  2  2.90%
117         NTRK1  2  2.90%
120          PAK7  2  2.90%
125        PDGFRB  2  2.90%
127       PIK3C2G  2  2.90%
132        PIK3CG  2  2.90%
136         POLD1  2  2.90%
139       PPP2R1A  2  2.90%
140         PREX2  2  2.90%
146         RAD50  2  2.90%
159         SF3B1  2  2.90%
161         SMAD4  2  2.90%
164          SOS1  2  2.90%
169         STAG2  2  2.90%
172         STK19  2  2.90%
173           SYK  2  2.90%
178          TET1  2  2.90%
182          TP63  2  2.90%
188         ZRSR2  2  2.90%
2         ANKRD11  1  1.45%
4              AR  1  1.45%
5          ARID1A  1  1.45%
6          ARID1B  1  1.45%
10            ATR  1  1.45%
12            AXL  1  1.45%
13           BAP1  1  1.45%
14           BCL2  1  1.45%
16            BLM  1  1.45%
20           BRD4  1  1.45%
21          BRIP1  1  1.45%
22            BTK  1  1.45%
23           CALR  1  1.45%
25            CBL  1  1.45%
27          CD274  1  1.45%
28          CDK12  1  1.45%
29         CDKN1B  1  1.45%
31          CSDE1  1  1.45%
33           CUL3  1  1.45%
34           CYLD  1  1.45%
39         EIF1AX  1  1.45%
40           ELF3  1  1.45%
42          EPAS1  1  1.45%
43          EPHA3  1  1.45%
46          EPHB1  1  1.45%
50          ERCC4  1  1.45%
51            ERG  1  1.45%
52           ESR1  1  1.45%
55          FBXW7  1  1.45%
57         FIP1L1  1  1.45%
60          FOXA1  1  1.45%
61          FOXL2  1  1.45%
62          FUBP1  1  1.45%
64           GLI1  1  1.45%
66          GSK3B  1  1.45%
67          H3F3A  1  1.45%
69       HIST1H1C  1  1.45%
70          HLA-A  1  1.45%
71         HOXB13  1  1.45%
73           IGF1  1  1.45%
74          IGF1R  1  1.45%
75          IKZF1  1  1.45%
79           INSR  1  1.45%
81           JAK3  1  1.45%
82          KDM5A  1  1.45%
92          LATS1  1  1.45%
93          LATS2  1  1.45%
94           MDC1  1  1.45%
95           MDM2  1  1.45%
98           MLH1  1  1.45%
99            MLL  1  1.45%
104           MYC  1  1.45%
105          MYCN  1  1.45%
109        NFE2L2  1  1.45%
110        NOTCH1  1  1.45%
111        NOTCH2  1  1.45%
115          NRAS  1  1.45%
116          NSD1  1  1.45%
118         NTRK2  1  1.45%
121          PAX5  1  1.45%
123         PDCD1  1  1.45%
126         PDPK1  1  1.45%
128        PIK3C3  1  1.45%
130        PIK3CB  1  1.45%
131        PIK3CD  1  1.45%
133        PIK3R1  1  1.45%
135          PMS2  1  1.45%
138         PPM1D  1  1.45%
142        PTPN11  1  1.45%
145          RAC2  1  1.45%
147         RASA1  1  1.45%
149         RBM10  1  1.45%
150        RECQL4  1  1.45%
152        RICTOR  1  1.45%
155         RPTOR  1  1.45%
156         RRAGC  1  1.45%
157         RTEL1  1  1.45%
160          SLX4  1  1.45%
163           SMO  1  1.45%
165         SOX17  1  1.45%
166          SOX2  1  1.45%
167          SOX9  1  1.45%
168          SPEN  1  1.45%
170        STAT5A  1  1.45%
171         STK11  1  1.45%
174          TAP1  1  1.45%
175        TCF7L2  1  1.45%
176           TEK  1  1.45%
179          TET2  1  1.45%
180          TOP1  1  1.45%
183          TSC1  1  1.45%
184          TSC2  1  1.45%
185           VHL  1  1.45%
186          XIAP  1  1.45%</code></pre>
<pre class="r"><code>library(ggplot2)
library(ggpubr)
mskdat_cd$Sample &lt;- seq.int(nrow(mskdat_cd))
ggplot(data = mskdat_cd, aes(x =Fraction.Genome.Altered, y = Mutation.Count,color=Sample,na.rm = TRUE)) + 
  geom_point(size=2, shape=20,na.rm = TRUE) + stat_cor(method = &quot;pearson&quot;,label.sep=&quot; Pearson Correlation Coef.,&quot;,cor.coef.name = &quot;R&quot;,na.rm = TRUE)+
  scale_y_continuous(breaks = seq(0, 37, by = 5)) +scale_x_continuous(breaks = seq(0, 1, by = .1))+
  labs(title = &quot;Mutation Count Vs Fraction Genome Altered &quot;, x = &quot;Fraction Genome Altered&quot;, y = &quot;Mutation Count&quot;)</code></pre>
<pre><code>Warning: The following aesthetics were dropped during statistical transformation: colour
i This can happen when ggplot fails to infer the correct grouping structure in
  the data.
i Did you forget to specify a `group` aesthetic or to convert a numerical
  variable into a factor?</code></pre>
<p><img src="figure/msk_impact_result.Rmd/unnamed-chunk-10-1.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>library(ggplot2)
library(ggpubr)
mskdat_cd$Sample &lt;- seq.int(nrow(mskdat_cd))
ggplot(data = mskdat_cd, aes(x =Fraction.Genome.Altered, y = Mutation.Count,color=Sample,na.rm = TRUE)) + 
  geom_point(size=2, shape=20,na.rm = TRUE) + stat_cor(method = &quot;spearman&quot;,label.sep=&quot; Spearman Correlation Coef.,&quot;, cor.coef.name =&quot;rho&quot;,na.rm = TRUE)+
  scale_y_continuous(breaks = seq(0, 37, by = 5)) +scale_x_continuous(breaks = seq(0, 1, by = .1))+
  labs(title = &quot;Mutation Count Vs Fraction Genome Altered &quot;, x = &quot;Fraction Genome Altered&quot;, y = &quot;Mutation Count&quot;)</code></pre>
<pre><code>Warning: The following aesthetics were dropped during statistical transformation: colour
i This can happen when ggplot fails to infer the correct grouping structure in
  the data.
i Did you forget to specify a `group` aesthetic or to convert a numerical
  variable into a factor?</code></pre>
<p><img src="figure/msk_impact_result.Rmd/unnamed-chunk-11-1.png" width="672" style="display: block; margin: auto;" /></p>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span>
Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<pre class="r"><code>sessionInfo()</code></pre>
<pre><code>R version 4.1.3 (2022-03-10)
Platform: x86_64-w64-mingw32/x64 (64-bit)
Running under: Windows 10 x64 (build 19044)

Matrix products: default

locale:
[1] LC_COLLATE=English_United States.1252 
[2] LC_CTYPE=English_United States.1252   
[3] LC_MONETARY=English_United States.1252
[4] LC_NUMERIC=C                          
[5] LC_TIME=English_United States.1252    

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
[1] ggpubr_0.5.0    ggplot2_3.4.0   scales_1.2.1    workflowr_1.7.0

loaded via a namespace (and not attached):
 [1] tidyselect_1.1.2 xfun_0.30        bslib_0.3.1      purrr_0.3.4     
 [5] carData_3.0-5    colorspace_2.0-3 vctrs_0.5.1      generics_0.1.2  
 [9] htmltools_0.5.2  yaml_2.3.5       utf8_1.2.2       rlang_1.0.6     
[13] jquerylib_0.1.4  later_1.3.0      pillar_1.7.0     glue_1.6.2      
[17] withr_2.5.0      lifecycle_1.0.3  stringr_1.4.0    munsell_0.5.0   
[21] ggsignif_0.6.4   gtable_0.3.0     evaluate_0.15    labeling_0.4.2  
[25] knitr_1.40       callr_3.7.0      fastmap_1.1.0    httpuv_1.6.5    
[29] ps_1.6.0         fansi_1.0.3      highr_0.9        broom_0.7.12    
[33] Rcpp_1.0.8.3     backports_1.4.1  promises_1.2.0.1 jsonlite_1.8.0  
[37] abind_1.4-5      farver_2.1.0     fs_1.5.2         digest_0.6.29   
[41] stringi_1.7.6    rstatix_0.7.1    processx_3.5.3   dplyr_1.0.8     
[45] getPass_0.2-2    rprojroot_2.0.3  grid_4.1.3       cli_3.4.1       
[49] tools_4.1.3      magrittr_2.0.3   sass_0.4.1       tibble_3.1.6    
[53] car_3.1-1        tidyr_1.2.0      crayon_1.5.1     whisker_0.4     
[57] pkgconfig_2.0.3  ellipsis_0.3.2   rmarkdown_2.13   httr_1.4.2      
[61] rstudioapi_0.13  R6_2.5.1         git2r_0.30.1     compiler_4.1.3  </code></pre>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
https://docs.mathjax.org/en/latest/web/configuration.html. This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>




</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->

<script>
$(document).ready(function ()  {

    // temporarily add toc-ignore selector to headers for the consistency with Pandoc
    $('.unlisted.unnumbered').addClass('toc-ignore')

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_');
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
