<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />


<meta name="author" content="akh" />

<meta name="date" content="2022-10-27" />

<title>R output</title>

<script src="site_libs/header-attrs-2.13/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/font-awesome-5.1.0/css/all.css" rel="stylesheet" />
<link href="site_libs/font-awesome-5.1.0/css/v4-shims.css" rel="stylesheet" />

<link rel="icon" href="https://github.com/workflowr/workflowr-assets/raw/main/img/reproducible.png">
<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>



<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>









<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
details > summary > p:only-child {
  display: inline;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.tab('show');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-bs-toggle="collapse" data-target="#navbar" data-bs-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">myproject</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/Akhileshpandey87/myproject">
    <span class="fab fa-github"></span>
     
    Source code
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">



<h1 class="title toc-ignore">R output</h1>
<h4 class="author">akh</h4>
<h4 class="date">2022-10-27</h4>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span>
workflowr <span class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span
class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span> </a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2022-11-03
</p>
<p>
<strong>Checks:</strong> <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 6
<span class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span> 1
</p>
<p>
<strong>Knit directory:</strong> <code>myproject/</code> <span
class="glyphicon glyphicon-question-sign" aria-hidden="true"
title="This is the local directory in which the code in this file was executed.">
</span>
</p>
<p>
This reproducible <a href="https://rmarkdown.rstudio.com">R Markdown</a>
analysis was created with <a
  href="https://github.com/workflowr/workflowr">workflowr</a> (version
1.7.0). The <em>Checks</em> tab describes the reproducibility checks
that were applied when the results were created. The <em>Past
versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguptodate">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>R Markdown file:</strong> up-to-date
</a>
</p>
</div>
<div id="strongRMarkdownfilestronguptodate"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great! Since the R Markdown file has been committed to the Git
repository, you know the exact version of the code that produced these
results.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the
global environment can affect the analysis in your R Markdown file in
unknown ways. For reproduciblity it’s best to always run the code in an
empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed20221022code">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Seed:</strong>
<code>set.seed(20221022)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed20221022code"
class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(20221022)</code> was run prior to running
the code in the R Markdown file. Setting a seed ensures that any results
that rely on randomness, e.g. subsampling or permutations, are
reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Session information:</strong>
recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package
versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be
confident that you successfully produced the results during this
run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongFilepathsstrongabsolute">
<span class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span> <strong>File paths:</strong> absolute </a>
</p>
</div>
<div id="strongFilepathsstrongabsolute" class="panel-collapse collapse">
<div class="panel-body">
<p>
Using absolute paths to the files within your workflowr project makes it
difficult for you and others to run your code on a different machine.
Change the absolute path(s) below to the suggested relative path(s) to
make your code more reproducible.
</p>
<table class="table table-condensed table-hover">
<thead>
<tr>
<th style="text-align:left;">
absolute
</th>
<th style="text-align:left;">
relative
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
~/Desktop/myproject/data/pancan_pcawg_2020/data_clinical_sample.txt
</td>
<td style="text-align:left;">
data/pancan_pcawg_2020/data_clinical_sample.txt
</td>
</tr>
<tr>
<td style="text-align:left;">
~/Desktop/myproject/data/pancan_pcawg_2020/data_clinical_patient.txt
</td>
<td style="text-align:left;">
data/pancan_pcawg_2020/data_clinical_patient.txt
</td>
</tr>
<tr>
<td style="text-align:left;">
~/Desktop/myproject/data/pancan_pcawg_2020_clinical_data.txt
</td>
<td style="text-align:left;">
data/pancan_pcawg_2020_clinical_data.txt
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrongahrefhttpsgithubcomAkhileshpandey87myprojecttree86ab8b2fdf6102d8ebf1cf58d90130e31799a567targetblank86ab8b2a">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Repository version:</strong>
<a href="https://github.com/Akhileshpandey87/myproject/tree/86ab8b2fdf6102d8ebf1cf58d90130e31799a567" target="_blank">86ab8b2</a>
</a>
</p>
</div>
<div
id="strongRepositoryversionstrongahrefhttpsgithubcomAkhileshpandey87myprojecttree86ab8b2fdf6102d8ebf1cf58d90130e31799a567targetblank86ab8b2a"
class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development
and connecting the code version to the results is critical for
reproducibility.
</p>
<p>
The results in this page were generated with repository version
<a href="https://github.com/Akhileshpandey87/myproject/tree/86ab8b2fdf6102d8ebf1cf58d90130e31799a567" target="_blank">86ab8b2</a>.
See the <em>Past versions</em> tab to see a history of the changes made
to the R Markdown and HTML files.
</p>
<p>
Note that you need to be careful to ensure that all relevant files for
the analysis have been committed to Git prior to generating the results
(you can use <code>wflow_publish</code> or
<code>wflow_git_commit</code>). workflowr only checks the R Markdown
file, but you know if there are other scripts or data files that it
depends on. Below is the status of the Git repository when the results
were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .Rhistory
    Ignored:    .Rproj.user/
    Ignored:    R-output_files/

Untracked files:
    Untracked:  data/pancan_pcawg_2020/
    Untracked:  data/pancan_pcawg_2020_clinical_data.txt
    Untracked:  site_libs/

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not
included in this status report because it is ok for generated content to
have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">

<p>
These are the previous versions of the repository in which changes were
made to the R Markdown (<code>analysis/R-output.Rmd</code>) and HTML
(<code>docs/R-output.html</code>) files. If you’ve configured a remote
Git repository (see <code>?wflow_git_remote</code>), click on the
hyperlinks in the table below to view the files as they were in that
past version.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/Akhileshpandey87/myproject/86ab8b2fdf6102d8ebf1cf58d90130e31799a567/docs/R-output.html" target="_blank">86ab8b2</a>
</td>
<td>
Akhileshpandey87
</td>
<td>
2022-11-03
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/Akhileshpandey87/myproject/blob/77ff564eddd9219da9fb01088a2387dbf3898e77/analysis/R-output.Rmd" target="_blank">77ff564</a>
</td>
<td>
Akhileshpandey87
</td>
<td>
2022-11-03
</td>
<td>
wflow_publish("analysis/R-output.Rmd")
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/Akhileshpandey87/myproject/blob/d42be1f8f648155b5010e289a7424f99d401917e/analysis/R-output.Rmd" target="_blank">d42be1f</a>
</td>
<td>
Akhileshpandey87
</td>
<td>
2022-11-01
</td>
<td>
wflow_git_commit(all = TRUE)
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/Akhileshpandey87/myproject/d42be1f8f648155b5010e289a7424f99d401917e/docs/R-output.html" target="_blank">d42be1f</a>
</td>
<td>
Akhileshpandey87
</td>
<td>
2022-11-01
</td>
<td>
wflow_git_commit(all = TRUE)
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/Akhileshpandey87/myproject/c8855f967f6de73376d1962e1010d48f3d055e9e/docs/R-output.html" target="_blank">c8855f9</a>
</td>
<td>
Akhileshpandey87
</td>
<td>
2022-11-01
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/Akhileshpandey87/myproject/6c06e13f3f4bece7f9a9ede58c6060a37da32e59/docs/R-output.html" target="_blank">6c06e13</a>
</td>
<td>
Akhileshpandey87
</td>
<td>
2022-11-01
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/Akhileshpandey87/myproject/blob/1b9282d87a189f7da1a6e46694d7a469f715c5aa/analysis/R-output.Rmd" target="_blank">1b9282d</a>
</td>
<td>
Akhileshpandey87
</td>
<td>
2022-11-01
</td>
<td>
wflow_publish("analysis/R-output.Rmd")
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/Akhileshpandey87/myproject/2071ca45d2399f504dc424a90981104d75e7e3f4/docs/R-output.html" target="_blank">2071ca4</a>
</td>
<td>
Akhileshpandey87
</td>
<td>
2022-10-30
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/Akhileshpandey87/myproject/blob/6e9c4677b5a07578c31819f40e4eeda653f9f5a3/analysis/R-output.Rmd" target="_blank">6e9c467</a>
</td>
<td>
Akhileshpandey87
</td>
<td>
2022-10-30
</td>
<td>
wflow_publish("analysis/R-output.Rmd")
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/Akhileshpandey87/myproject/5a81c513cf763bc9a1476a5385ad9ec911745955/docs/R-output.html" target="_blank">5a81c51</a>
</td>
<td>
Akhileshpandey87
</td>
<td>
2022-10-30
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/Akhileshpandey87/myproject/blob/dddd215f6007c55c07ff2214377eb7987e5d5ca2/analysis/R-output.Rmd" target="_blank">dddd215</a>
</td>
<td>
Akhileshpandey87
</td>
<td>
2022-10-30
</td>
<td>
wflow_publish("analysis/R-output.Rmd")
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<div id="r-markdown" class="section level2">
<h2>R Markdown</h2>
<p>This is an R Markdown document. Markdown is a simple formatting
syntax for authoring HTML, PDF, and MS Word documents. For more details
on using R Markdown see <a href="http://rmarkdown.rstudio.com"
class="uri">http://rmarkdown.rstudio.com</a>.</p>
<p>When you click the <strong>Knit</strong> button a document will be
generated that includes both content as well as the output of any
embedded R code chunks within the document. You can embed an R code
chunk like this:</p>
<p>Note that the <code>echo = FALSE</code> parameter was added to the
code chunk to prevent printing of the R code that generated the
plot.</p>
<pre class="r"><code>dat &lt;- read.delim(&quot;~/Desktop/myproject/data/pancan_pcawg_2020/data_clinical_sample.txt&quot;,header=TRUE,stringsAsFactor = TRUE,comment.char = &quot;#&quot;)
  head(dat)</code></pre>
<pre><code>  SAMPLE_ID PATIENT_ID PROJECT_CODE STAGE GRADE ICD_10    ORGAN_SYSTEM
1    SP1003      DO496      BLCA-US  &lt;NA&gt;  &lt;NA&gt;  C67.9 URINARY BLADDER
2   SP10084     DO4766      BRCA-US  &lt;NA&gt;  &lt;NA&gt;  C50.9          BREAST
3    SP1009      DO498      BLCA-US  &lt;NA&gt;  &lt;NA&gt;  C67.9 URINARY BLADDER
4   SP10150     DO4796      BRCA-US  &lt;NA&gt;  &lt;NA&gt;  C50.9          BREAST
5  SP101515    DO46325        OV-AU   III    G3  C48.2           OVARY
6  SP101519    DO46327        OV-AU   III    G3    C56           OVARY
  HISTOLOGY_ABBREVIATION                        HISTOLOGY HISTOLOGY_TIER1
1            Bladder-TCC  Papillary trans. cell carcinoma        ENDODERM
2         Breast-AdenoCA Infiltrating duct carcinoma, NOS        ECTODERM
3            Bladder-TCC Transitional cell carcinoma, NOS        ENDODERM
4         Breast-AdenoCA Infiltrating duct carcinoma, NOS        ECTODERM
5          Ovary-AdenoCA   Serous cystadenocarcinoma, NOS        MESODERM
6          Ovary-AdenoCA   Serous cystadenocarcinoma, NOS        MESODERM
  HISTOLOGY_TIER2             HISTOLOGY_TIER3
1         Bladder Transitional cell carcinoma
2          Breast              Adenocarcinoma
3         Bladder Transitional cell carcinoma
4          Breast              Adenocarcinoma
5           Ovary              Adenocarcinoma
6           Ovary              Adenocarcinoma
                         HISTOLOGY_TIER4 TUMOR_SAMPLE_HISTOLOGY_CODE
1 Transitional cell carcinoma, papillary                      8130/3
2            Infiltrating duct carcinoma                      8500/3
3            Transitional cell carcinoma                      8120/3
4            Infiltrating duct carcinoma                      8500/3
5              Serous cystadenocarcinoma                      8441/3
6              Serous cystadenocarcinoma                      8441/3
  CELLULARITY PURITY PLOIDY PURITY_CONFUGURATION    WGD SAMPLE_TYPE
1          NA  0.613  3.228                0.016    wgd     Primary
2          90  0.693  3.986                0.002    wgd     Primary
3          80  0.827  2.002                0.028 no_wgd     Primary
4          80  0.727  3.359                0.031    wgd     Primary
5          73  0.540  2.112                0.023 no_wgd     Primary
6          77  0.478  2.623                0.010    wgd     Primary
     CANCER_TYPE             CANCER_TYPE_DETAILED ONCOTREE_CODE ICGC_SAMPLE_ID
1 Bladder Cancer  Bladder Squamous Cell Carcinoma          BLSC         SA4557
2  Breast Cancer Breast Invasive Ductal Carcinoma           IDC        SA48667
3 Bladder Cancer  Bladder Squamous Cell Carcinoma          BLSC         SA4666
4  Breast Cancer Breast Invasive Ductal Carcinoma           IDC        SA49610
5 Ovarian Cancer            Serous Ovarian Cancer           SOC       SA505244
6 Ovarian Cancer            Serous Ovarian Cancer           SOC       SA505270
  SEQUENCING_TYPE SAMPLE_CLASS ANCESTRY_PRIMARY TMB_NONSYNONYMOUS
1             WGS        Tumor              EUR          8.433333
2             WGS        Tumor              EUR          2.066667
3             WGS        Tumor              EUR          4.000000
4             WGS        Tumor              EUR          2.500000
5             WGS        Tumor              EUR          2.066667
6             WGS        Tumor              EUR          2.100000</code></pre>
<pre class="r"><code>tail(dat)</code></pre>
<pre><code>     SAMPLE_ID PATIENT_ID PROJECT_CODE STAGE GRADE ICD_10 ORGAN_SYSTEM
2917   SP99329    DO45301      LIRI-JP     2  &lt;NA&gt;    C22        LIVER
2918   SP99333    DO45303      LIRI-JP     3  &lt;NA&gt;    C22        LIVER
2919   SP99337    DO45305      LIRI-JP     4  &lt;NA&gt;    C22        LIVER
2920   SP99341    DO45307      LIRI-JP     4  &lt;NA&gt;    C22        LIVER
2921   SP99345    DO45309      LIRI-JP     3  &lt;NA&gt;    C22        LIVER
2922    SP9979     DO4719      BRCA-US  &lt;NA&gt;  &lt;NA&gt;  C50.9       BREAST
     HISTOLOGY_ABBREVIATION                        HISTOLOGY HISTOLOGY_TIER1
2917        Biliary-AdenoCA               Cholangiocarcinoma        ENDODERM
2918        Biliary-AdenoCA               Cholangiocarcinoma        ENDODERM
2919        Biliary-AdenoCA               Cholangiocarcinoma        ENDODERM
2920        Biliary-AdenoCA               Cholangiocarcinoma        ENDODERM
2921        Biliary-AdenoCA               Cholangiocarcinoma        ENDODERM
2922         Breast-AdenoCA Infiltrating duct carcinoma, NOS        ECTODERM
     HISTOLOGY_TIER2 HISTOLOGY_TIER3             HISTOLOGY_TIER4
2917         Biliary  Adenocarcinoma          Cholangiocarcinoma
2918         Biliary  Adenocarcinoma          Cholangiocarcinoma
2919         Biliary  Adenocarcinoma          Cholangiocarcinoma
2920         Biliary  Adenocarcinoma          Cholangiocarcinoma
2921         Biliary  Adenocarcinoma          Cholangiocarcinoma
2922          Breast  Adenocarcinoma Infiltrating duct carcinoma
     TUMOR_SAMPLE_HISTOLOGY_CODE CELLULARITY PURITY PLOIDY PURITY_CONFUGURATION
2917                      8160/3          NA  0.310  1.917                0.015
2918                      8160/3          NA  0.680  1.849                0.001
2919                      8160/3          NA  0.796  1.843                0.006
2920                      8160/3          NA  0.330  2.108                0.084
2921                      8160/3          NA  0.221  1.927                0.090
2922                      8500/3          NA  0.447  2.152                0.023
        WGD SAMPLE_TYPE          CANCER_TYPE             CANCER_TYPE_DETAILED
2917 no_wgd     Primary Hepatobiliary Cancer               Cholangiocarcinoma
2918 no_wgd     Primary Hepatobiliary Cancer               Cholangiocarcinoma
2919 no_wgd     Primary Hepatobiliary Cancer               Cholangiocarcinoma
2920 no_wgd     Primary Hepatobiliary Cancer               Cholangiocarcinoma
2921 no_wgd     Primary Hepatobiliary Cancer               Cholangiocarcinoma
2922 no_wgd     Primary        Breast Cancer Breast Invasive Ductal Carcinoma
     ONCOTREE_CODE ICGC_SAMPLE_ID SEQUENCING_TYPE SAMPLE_CLASS ANCESTRY_PRIMARY
2917          CHOL       SA501614             WGS        Tumor              ASN
2918          CHOL       SA501618             WGS        Tumor              ASN
2919          CHOL       SA501622             WGS        Tumor              ASN
2920          CHOL       SA501626             WGS        Tumor              ASN
2921          CHOL       SA501630             WGS        Tumor              ASN
2922           IDC       SA491910             WGS        Tumor             &lt;NA&gt;
     TMB_NONSYNONYMOUS
2917         2.1000000
2918         0.7333333
2919         0.8666667
2920         1.1333333
2921         0.9333333
2922         0.9333333</code></pre>
<pre class="r"><code>class(dat)</code></pre>
<pre><code>[1] &quot;data.frame&quot;</code></pre>
<pre class="r"><code>dim(dat)</code></pre>
<pre><code>[1] 2922   28</code></pre>
<pre class="r"><code>typeof(dat)</code></pre>
<pre><code>[1] &quot;list&quot;</code></pre>
<pre class="r"><code>str(dat)</code></pre>
<pre><code>&#39;data.frame&#39;:   2922 obs. of  28 variables:
 $ SAMPLE_ID                  : Factor w/ 2922 levels &quot;SP1003&quot;,&quot;SP10084&quot;,..: 1 2 3 4 5 6 7 8 9 10 ...
 $ PATIENT_ID                 : Factor w/ 2583 levels &quot;DO1000&quot;,&quot;DO1001&quot;,..: 1828 1611 1836 1624 1431 1432 1433 1434 1434 1435 ...
 $ PROJECT_CODE               : Factor w/ 47 levels &quot;BLCA-US&quot;,&quot;BOCA-UK&quot;,..: 1 5 1 5 31 31 31 31 31 31 ...
 $ STAGE                      : Factor w/ 121 levels &quot;1&quot;,&quot;1a&quot;,&quot;1b&quot;,..: NA NA NA NA 21 21 21 21 NA 21 ...
 $ GRADE                      : Factor w/ 26 levels &quot;2 - Moderately differentiated&quot;,..: NA NA NA NA 14 14 14 14 NA 14 ...
 $ ICD_10                     : Factor w/ 136 levels &quot;0&quot;,&quot;C01&quot;,&quot;C02.9&quot;,..: 107 86 107 86 69 93 93 93 93 93 ...
 $ ORGAN_SYSTEM               : Factor w/ 23 levels &quot;BLOOD, BONE MARROW, &amp; HEMATOPOIETIC SYS&quot;,..: 22 5 22 5 15 15 15 15 NA 15 ...
 $ HISTOLOGY_ABBREVIATION     : Factor w/ 39 levels &quot;Biliary-AdenoCA&quot;,..: 2 8 2 8 30 30 30 30 NA 30 ...
 $ HISTOLOGY                  : Factor w/ 68 levels &quot;Acinar cell carcinoma&quot;,..: 57 32 66 32 64 64 64 7 7 64 ...
 $ HISTOLOGY_TIER1            : Factor w/ 4 levels &quot;ECTODERM&quot;,&quot;ENDODERM&quot;,..: 2 1 2 1 3 3 3 3 NA 3 ...
 $ HISTOLOGY_TIER2            : Factor w/ 21 levels &quot;Biliary&quot;,&quot;Bladder&quot;,..: 2 4 2 4 15 15 15 15 NA 15 ...
 $ HISTOLOGY_TIER3            : Factor w/ 25 levels &quot;Acute myeloid leukemia&quot;,..: 25 2 25 2 2 2 2 2 NA 2 ...
 $ HISTOLOGY_TIER4            : Factor w/ 66 levels &quot;Acinar cell carcinoma&quot;,..: 66 38 65 38 61 61 61 4 NA 61 ...
 $ TUMOR_SAMPLE_HISTOLOGY_CODE: Factor w/ 66 levels &quot;8022/3&quot;,&quot;8050/3&quot;,..: 6 28 5 28 24 24 24 7 NA 24 ...
 $ CELLULARITY                : int  NA 90 80 80 73 77 90 84 99 82 ...
 $ PURITY                     : num  0.613 0.693 0.827 0.727 0.54 0.478 0.71 0.789 NA 0.498 ...
 $ PLOIDY                     : num  3.23 3.99 2 3.36 2.11 ...
 $ PURITY_CONFUGURATION       : num  0.016 0.002 0.028 0.031 0.023 0.01 0.016 0.012 NA 0.018 ...
 $ WGD                        : Factor w/ 2 levels &quot;no_wgd&quot;,&quot;wgd&quot;: 2 2 1 2 1 2 2 1 NA 2 ...
 $ SAMPLE_TYPE                : Factor w/ 4 levels &quot;Cell line&quot;,&quot;Local Recurrence&quot;,..: 4 4 4 4 4 4 4 4 2 4 ...
 $ CANCER_TYPE                : Factor w/ 30 levels &quot;Acute myeloid leukemia&quot;,..: 2 4 2 4 23 23 23 23 23 23 ...
 $ CANCER_TYPE_DETAILED       : Factor w/ 74 levels &quot;Acinar Cell Carcinoma of the Pancreas&quot;,..: 6 7 6 7 70 70 70 58 58 70 ...
 $ ONCOTREE_CODE              : Factor w/ 71 levels &quot;ADMA&quot;,&quot;AMBL&quot;,..: 5 30 5 30 65 65 65 53 53 65 ...
 $ ICGC_SAMPLE_ID             : Factor w/ 2583 levels &quot;SA100232&quot;,&quot;SA100712&quot;,..: 748 789 767 852 1030 1033 1035 1036 NA 1037 ...
 $ SEQUENCING_TYPE            : Factor w/ 2 levels &quot;WGS&quot;,&quot;WGS+RNA-Seq&quot;: 1 1 1 1 1 1 1 1 NA 1 ...
 $ SAMPLE_CLASS               : Factor w/ 2 levels &quot;Cell line&quot;,&quot;Tumor&quot;: 2 2 2 2 2 2 2 2 2 2 ...
 $ ANCESTRY_PRIMARY           : Factor w/ 5 levels &quot;AFR&quot;,&quot;AMR&quot;,&quot;ASN&quot;,..: 4 4 4 4 4 4 4 3 NA 4 ...
 $ TMB_NONSYNONYMOUS          : num  8.43 2.07 4 2.5 2.07 ...</code></pre>
<pre class="r"><code>table(dat$WGD)</code></pre>
<pre><code>
no_wgd    wgd 
  1900    803 </code></pre>
<pre class="r"><code>table(dat$WGD,dat$SEQUENCING_TYPE)</code></pre>
<pre><code>        
          WGS WGS+RNA-Seq
  no_wgd 1799           4
  wgd     777           3</code></pre>
<pre class="r"><code>plot(dat$WGD)</code></pre>
<p><img src="figure/R-output.Rmd/unnamed-chunk-10-1.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>plot(dat$WGD,dat$SEQUENCING_TYPE)</code></pre>
<p><img src="figure/R-output.Rmd/unnamed-chunk-11-1.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>summary(dat)</code></pre>
<pre><code>    SAMPLE_ID      PATIENT_ID    PROJECT_CODE      STAGE     
 SP1003  :   1   DO51954:  10   LIRI-JP: 260   unknown: 148  
 SP10084 :   1   DO51964:   9   PBCA-DE: 230   2      : 119  
 SP1009  :   1   DO52509:   6   PACA-AU: 166   III    : 104  
 SP10150 :   1   DO52510:   6   CLLE-ES: 158   3      :  75  
 SP101515:   1   DO52512:   6   PACA-CA: 143   A      :  66  
 SP101519:   1   DO52513:   6   RECA-EU: 123   (Other):1112  
 (Other) :2916   (Other):2879   (Other):1842   NA&#39;s   :1298  
                           GRADE          ICD_10    
 G3                           :  75   C61    : 275  
 G2                           :  52   C22    : 260  
 II                           :  50   C25.0  : 212  
 2 - Moderately differentiated:  37   C91.1  : 158  
 G4                           :  37   0      : 142  
 (Other)                      : 148   C71    : 141  
 NA&#39;s                         :2523   (Other):1734  
                                                                ORGAN_SYSTEM 
 LIVER                                                                : 336  
 PANCREAS                                                             : 313  
 BRAIN, &amp; CRANIAL NERVES, &amp; SPINAL CORD, (EXCL. VENTRICLE, CEREBELLUM): 287  
 BREAST                                                               : 211  
 PROSTATE GLAND                                                       : 199  
 (Other)                                                              :1237  
 NA&#39;s                                                                 : 339  
    HISTOLOGY_ABBREVIATION
 Liver-HCC     : 314      
 Panc-AdenoCA  : 232      
 Prost-AdenoCA : 199      
 Breast-AdenoCA: 195      
 Kidney-RCC    : 143      
 (Other)       :1500      
 NA&#39;s          : 339      
                                                   HISTOLOGY   
 Adenocarcinoma, NOS                                    : 467  
 Infiltrating duct carcinoma, NOS                       : 402  
 Hepatocellular carcinoma, NOS                          : 315  
 Clear cell adenocarcinoma, NOS                         : 160  
 Chronic lymphocytic leukemia/small lymphocytic lymphoma: 158  
 (Other)                                                :1344  
 NA&#39;s                                                   :  76  
     HISTOLOGY_TIER1 HISTOLOGY_TIER2
 ECTODERM    : 211   Liver   : 314  
 ENDODERM    :1232   Pancreas: 313  
 MESODERM    : 746   CNS     : 287  
 NEURAL_CREST: 394   Breast  : 211  
 NA&#39;s        : 339   Prostate: 199  
                     (Other) :1259  
                     NA&#39;s    : 339  
                                HISTOLOGY_TIER3
 Adenocarcinoma                         :1118  
 Hepatocellular carcinoma               : 314  
 Renal cell carcinoma (proximal tubules): 143  
 Medulloblastoma                        : 141  
 Squamous cell carcinoma                : 121  
 (Other)                                : 746  
 NA&#39;s                                   : 339  
                    HISTOLOGY_TIER4 TUMOR_SAMPLE_HISTOLOGY_CODE
 Adenocarcinoma             : 426   8140/3 : 430               
 Hepatocellular carcinoma   : 305   8500/3 : 348               
 Infiltrating duct carcinoma: 191   8170/3 : 305               
 Pancreatic ductal carcinoma: 157   8070/3 : 120               
 Serous cystadenocarcinoma  : 120   8441/3 : 120               
 (Other)                    :1384   (Other):1260               
 NA&#39;s                       : 339   NA&#39;s   : 339               
  CELLULARITY         PURITY           PLOIDY      PURITY_CONFUGURATION
 Min.   : 12.00   Min.   :0.1300   Min.   :1.283   Min.   :0.0000      
 1st Qu.: 71.00   1st Qu.:0.4730   1st Qu.:1.926   1st Qu.:0.0060      
 Median : 84.00   Median :0.6590   Median :2.000   Median :0.0130      
 Mean   : 80.63   Mean   :0.6401   Mean   :2.365   Mean   :0.0211      
 3rd Qu.: 95.00   3rd Qu.:0.8240   3rd Qu.:2.795   3rd Qu.:0.0220      
 Max.   :100.00   Max.   :1.0000   Max.   :6.217   Max.   :0.4180      
 NA&#39;s   :2080     NA&#39;s   :219      NA&#39;s   :219     NA&#39;s   :219         
     WGD                 SAMPLE_TYPE                   CANCER_TYPE  
 no_wgd:1900   Cell line       :   1   Pancreatic Cancer     : 381  
 wgd   : 803   Local Recurrence:  41   Hepatobiliary Cancer  : 358  
 NA&#39;s  : 219   Metastasis      : 147   Prostate Cancer       : 275  
               Primary         :2731   Renal Cell Carcinoma  : 235  
               NA&#39;s            :   2   Breast Cancer         : 213  
                                       Esophagogastric Cancer: 165  
                                       (Other)               :1295  
                                              CANCER_TYPE_DETAILED
 Hepatocellular Carcinoma                               : 315     
 Pancreatic Adenocarcinoma                              : 283     
 Prostate Adenocarcinoma                                : 275     
 Breast Invasive Ductal Carcinoma                       : 180     
 Renal Clear Cell Carcinoma                             : 160     
 Chronic Lymphocytic Leukemia/Small Lymphocytic Lymphoma: 158     
 (Other)                                                :1551     
 ONCOTREE_CODE   ICGC_SAMPLE_ID    SEQUENCING_TYPE    SAMPLE_CLASS 
 HCC    : 323   SA100232:   1   WGS        :2576   Cell line:   1  
 PAAD   : 285   SA100712:   1   WGS+RNA-Seq:   7   Tumor    :2919  
 PRAD   : 275   SA100868:   1   NA&#39;s       : 339   NA&#39;s     :   2  
 IDC    : 180   SA101371:   1                                      
 CCRCC  : 160   SA10195 :   1                                      
 CLLSLL : 158   (Other) :2578                                      
 (Other):1541   NA&#39;s    : 339                                      
 ANCESTRY_PRIMARY TMB_NONSYNONYMOUS
 AFR : 124        Min.   :  0.000  
 AMR :  26        1st Qu.:  0.700  
 ASN : 415        Median :  1.500  
 EUR :1968        Mean   :  3.507  
 SAN :  36        3rd Qu.:  2.800  
 NA&#39;s: 353        Max.   :402.033  
                  NA&#39;s   :239      </code></pre>
<pre class="r"><code>str(dat)</code></pre>
<pre><code>&#39;data.frame&#39;:   2922 obs. of  28 variables:
 $ SAMPLE_ID                  : Factor w/ 2922 levels &quot;SP1003&quot;,&quot;SP10084&quot;,..: 1 2 3 4 5 6 7 8 9 10 ...
 $ PATIENT_ID                 : Factor w/ 2583 levels &quot;DO1000&quot;,&quot;DO1001&quot;,..: 1828 1611 1836 1624 1431 1432 1433 1434 1434 1435 ...
 $ PROJECT_CODE               : Factor w/ 47 levels &quot;BLCA-US&quot;,&quot;BOCA-UK&quot;,..: 1 5 1 5 31 31 31 31 31 31 ...
 $ STAGE                      : Factor w/ 121 levels &quot;1&quot;,&quot;1a&quot;,&quot;1b&quot;,..: NA NA NA NA 21 21 21 21 NA 21 ...
 $ GRADE                      : Factor w/ 26 levels &quot;2 - Moderately differentiated&quot;,..: NA NA NA NA 14 14 14 14 NA 14 ...
 $ ICD_10                     : Factor w/ 136 levels &quot;0&quot;,&quot;C01&quot;,&quot;C02.9&quot;,..: 107 86 107 86 69 93 93 93 93 93 ...
 $ ORGAN_SYSTEM               : Factor w/ 23 levels &quot;BLOOD, BONE MARROW, &amp; HEMATOPOIETIC SYS&quot;,..: 22 5 22 5 15 15 15 15 NA 15 ...
 $ HISTOLOGY_ABBREVIATION     : Factor w/ 39 levels &quot;Biliary-AdenoCA&quot;,..: 2 8 2 8 30 30 30 30 NA 30 ...
 $ HISTOLOGY                  : Factor w/ 68 levels &quot;Acinar cell carcinoma&quot;,..: 57 32 66 32 64 64 64 7 7 64 ...
 $ HISTOLOGY_TIER1            : Factor w/ 4 levels &quot;ECTODERM&quot;,&quot;ENDODERM&quot;,..: 2 1 2 1 3 3 3 3 NA 3 ...
 $ HISTOLOGY_TIER2            : Factor w/ 21 levels &quot;Biliary&quot;,&quot;Bladder&quot;,..: 2 4 2 4 15 15 15 15 NA 15 ...
 $ HISTOLOGY_TIER3            : Factor w/ 25 levels &quot;Acute myeloid leukemia&quot;,..: 25 2 25 2 2 2 2 2 NA 2 ...
 $ HISTOLOGY_TIER4            : Factor w/ 66 levels &quot;Acinar cell carcinoma&quot;,..: 66 38 65 38 61 61 61 4 NA 61 ...
 $ TUMOR_SAMPLE_HISTOLOGY_CODE: Factor w/ 66 levels &quot;8022/3&quot;,&quot;8050/3&quot;,..: 6 28 5 28 24 24 24 7 NA 24 ...
 $ CELLULARITY                : int  NA 90 80 80 73 77 90 84 99 82 ...
 $ PURITY                     : num  0.613 0.693 0.827 0.727 0.54 0.478 0.71 0.789 NA 0.498 ...
 $ PLOIDY                     : num  3.23 3.99 2 3.36 2.11 ...
 $ PURITY_CONFUGURATION       : num  0.016 0.002 0.028 0.031 0.023 0.01 0.016 0.012 NA 0.018 ...
 $ WGD                        : Factor w/ 2 levels &quot;no_wgd&quot;,&quot;wgd&quot;: 2 2 1 2 1 2 2 1 NA 2 ...
 $ SAMPLE_TYPE                : Factor w/ 4 levels &quot;Cell line&quot;,&quot;Local Recurrence&quot;,..: 4 4 4 4 4 4 4 4 2 4 ...
 $ CANCER_TYPE                : Factor w/ 30 levels &quot;Acute myeloid leukemia&quot;,..: 2 4 2 4 23 23 23 23 23 23 ...
 $ CANCER_TYPE_DETAILED       : Factor w/ 74 levels &quot;Acinar Cell Carcinoma of the Pancreas&quot;,..: 6 7 6 7 70 70 70 58 58 70 ...
 $ ONCOTREE_CODE              : Factor w/ 71 levels &quot;ADMA&quot;,&quot;AMBL&quot;,..: 5 30 5 30 65 65 65 53 53 65 ...
 $ ICGC_SAMPLE_ID             : Factor w/ 2583 levels &quot;SA100232&quot;,&quot;SA100712&quot;,..: 748 789 767 852 1030 1033 1035 1036 NA 1037 ...
 $ SEQUENCING_TYPE            : Factor w/ 2 levels &quot;WGS&quot;,&quot;WGS+RNA-Seq&quot;: 1 1 1 1 1 1 1 1 NA 1 ...
 $ SAMPLE_CLASS               : Factor w/ 2 levels &quot;Cell line&quot;,&quot;Tumor&quot;: 2 2 2 2 2 2 2 2 2 2 ...
 $ ANCESTRY_PRIMARY           : Factor w/ 5 levels &quot;AFR&quot;,&quot;AMR&quot;,&quot;ASN&quot;,..: 4 4 4 4 4 4 4 3 NA 4 ...
 $ TMB_NONSYNONYMOUS          : num  8.43 2.07 4 2.5 2.07 ...</code></pre>
<p>library(ggplot2)</p>
<pre class="r"><code>library(ggplot2)
ggplot(data = dat, mapping = aes(x = PURITY, y = PLOIDY)) +
  geom_point()</code></pre>
<pre><code>Warning: Removed 219 rows containing missing values (geom_point).</code></pre>
<p><img src="figure/R-output.Rmd/unnamed-chunk-15-1.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>ggplot(dat, aes(x=PURITY)) + geom_histogram(binwidth=.1)</code></pre>
<pre><code>Warning: Removed 219 rows containing non-finite values (stat_bin).</code></pre>
<p><img src="figure/R-output.Rmd/unnamed-chunk-16-1.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>ggplot(dat, aes(x=PURITY)) + geom_histogram(binwidth=.1,colour=&quot;black&quot;, fill=&quot;white&quot;)</code></pre>
<pre><code>Warning: Removed 219 rows containing non-finite values (stat_bin).</code></pre>
<p><img src="figure/R-output.Rmd/unnamed-chunk-17-1.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>ggplot(dat, aes(x=PURITY)) + geom_density()</code></pre>
<pre><code>Warning: Removed 219 rows containing non-finite values (stat_density).</code></pre>
<p><img src="figure/R-output.Rmd/unnamed-chunk-18-1.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>ggplot(dat, aes(x=PURITY)) + geom_histogram(aes(y=..density..),      # Histogram with density instead of count on y-axis
                   binwidth=.1,
                   colour=&quot;black&quot;, fill=&quot;white&quot;)+ geom_density(alpha=.2, fill=&quot;#FF6666&quot;)</code></pre>
<pre><code>Warning: Removed 219 rows containing non-finite values (stat_bin).</code></pre>
<pre><code>Warning: Removed 219 rows containing non-finite values (stat_density).</code></pre>
<p><img src="figure/R-output.Rmd/unnamed-chunk-19-1.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>ggplot(dat, aes(x=PURITY)) + geom_histogram(
                   binwidth=.1,
                   colour=&quot;black&quot;, fill=&quot;white&quot;)+ geom_vline(aes(xintercept=mean(PURITY, na.rm=T)),   # Ignore NA values for mean
               color=&quot;red&quot;, linetype=&quot;dashed&quot;, size=1)</code></pre>
<pre><code>Warning: Removed 219 rows containing non-finite values (stat_bin).</code></pre>
<p><img src="figure/R-output.Rmd/unnamed-chunk-20-1.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>dat1 &lt;- read.delim(&quot;~/Desktop/myproject/data/pancan_pcawg_2020/data_clinical_patient.txt&quot;,header=TRUE,stringsAsFactor = TRUE,comment.char = &quot;#&quot;)
  head(dat1)</code></pre>
<pre><code>  PATIENT_ID    SEX  OS_STATUS OS_MONTHS FIRST_THERAPY FIRST.THERAPY_RESPONSE
1      DO472   Male   0:LIVING        NA          &lt;NA&gt;                   &lt;NA&gt;
2      DO477   Male 1:DECEASED        NA          &lt;NA&gt;                   &lt;NA&gt;
3      DO479   Male   0:LIVING        NA          &lt;NA&gt;                   &lt;NA&gt;
4      DO483 Female   0:LIVING        NA          &lt;NA&gt;                   &lt;NA&gt;
5      DO496 Female   0:LIVING         0          &lt;NA&gt;                   &lt;NA&gt;
6      DO498   Male   0:LIVING         0          &lt;NA&gt;                   &lt;NA&gt;
  AGE TOBACCO_SMOKING_HISTORY_INDICATOR TOBACCO_SMOKING_INTENSITY ALCOHOL
1  53                              &lt;NA&gt;                        NA    &lt;NA&gt;
2  79                              &lt;NA&gt;                        NA    &lt;NA&gt;
3  34                              &lt;NA&gt;                        NA    &lt;NA&gt;
4  67                              &lt;NA&gt;                        NA    &lt;NA&gt;
5  53                              &lt;NA&gt;                        NA    &lt;NA&gt;
6  84                              &lt;NA&gt;                        NA    &lt;NA&gt;
  ALCOHOL_HISTORY_INTENSITY
1                      &lt;NA&gt;
2                      &lt;NA&gt;
3                      &lt;NA&gt;
4                      &lt;NA&gt;
5                      &lt;NA&gt;
6                      &lt;NA&gt;</code></pre>
<pre class="r"><code>ggplot(data=dat1, aes(x=AGE)) +
    geom_bar(width=5,colour=&quot;black&quot;,fill=&quot;blue&quot;)</code></pre>
<pre><code>Warning: Removed 123 rows containing non-finite values (stat_count).</code></pre>
<pre><code>Warning: position_stack requires non-overlapping x intervals</code></pre>
<p><img src="figure/R-output.Rmd/unnamed-chunk-22-1.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code># Add title, narrower bars, fill color, and change axis labels
ggplot(data=dat1, aes(x=AGE)) +
    geom_bar(width=1,colour=&quot;black&quot;,fill=&quot;blue&quot;,stat=&quot;count&quot;)+ 
    xlab(&quot;Age of patients&quot;) +
    ggtitle(&quot;Age distribution of patients&quot;)</code></pre>
<pre><code>Warning: Removed 123 rows containing non-finite values (stat_count).</code></pre>
<p><img src="figure/R-output.Rmd/unnamed-chunk-23-1.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code># Mutation count
ggplot(data=dat1, aes(x=AGE)) +
    geom_bar(width=1,colour=&quot;black&quot;,fill=&quot;blue&quot;,stat=&quot;count&quot;)+ 
    guides(fill=FALSE) +
    xlab(&quot;Age of patients&quot;) +
    ggtitle(&quot;Age distribution of patients&quot;)</code></pre>
<pre><code>Warning: `guides(&lt;scale&gt; = FALSE)` is deprecated. Please use `guides(&lt;scale&gt; =
&quot;none&quot;)` instead.</code></pre>
<pre><code>Warning: Removed 123 rows containing non-finite values (stat_count).</code></pre>
<p><img src="figure/R-output.Rmd/unnamed-chunk-24-1.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>dat2 &lt;- read.delim(&quot;~/Desktop/myproject/data/pancan_pcawg_2020_clinical_data.txt&quot;,header=TRUE,stringsAsFactor = TRUE,comment.char = &quot;#&quot;)
  head(dat2)</code></pre>
<pre><code>           Study.ID Patient.ID Sample.ID Age.at.Diagnosis Alcohol
1 pancan_pcawg_2020      DO496    SP1003               53    &lt;NA&gt;
2 pancan_pcawg_2020     DO4766   SP10084               64    &lt;NA&gt;
3 pancan_pcawg_2020      DO498    SP1009               84    &lt;NA&gt;
4 pancan_pcawg_2020     DO4796   SP10150               48    &lt;NA&gt;
5 pancan_pcawg_2020    DO46325  SP101515               54    &lt;NA&gt;
6 pancan_pcawg_2020    DO46327  SP101519               54    &lt;NA&gt;
  Alcohol.History.Intensity Primary.Ancestry    Cancer.Type
1                      &lt;NA&gt;              EUR Bladder Cancer
2                      &lt;NA&gt;              EUR  Breast Cancer
3                      &lt;NA&gt;              EUR Bladder Cancer
4                      &lt;NA&gt;              EUR  Breast Cancer
5                      &lt;NA&gt;              EUR Ovarian Cancer
6                      &lt;NA&gt;              EUR Ovarian Cancer
              Cancer.Type.Detailed Cellularity First.Therapy.Response
1  Bladder Squamous Cell Carcinoma          NA                   &lt;NA&gt;
2 Breast Invasive Ductal Carcinoma          90                   &lt;NA&gt;
3  Bladder Squamous Cell Carcinoma          80                   &lt;NA&gt;
4 Breast Invasive Ductal Carcinoma          80                   &lt;NA&gt;
5            Serous Ovarian Cancer          73                   &lt;NA&gt;
6            Serous Ovarian Cancer          77                   &lt;NA&gt;
  First.Therapy Grade                        Histology Histology.Abbreviation
1          &lt;NA&gt;  &lt;NA&gt;  Papillary trans. cell carcinoma            Bladder-TCC
2          &lt;NA&gt;  &lt;NA&gt; Infiltrating duct carcinoma, NOS         Breast-AdenoCA
3          &lt;NA&gt;  &lt;NA&gt; Transitional cell carcinoma, NOS            Bladder-TCC
4          &lt;NA&gt;  &lt;NA&gt; Infiltrating duct carcinoma, NOS         Breast-AdenoCA
5          &lt;NA&gt;    G3   Serous cystadenocarcinoma, NOS          Ovary-AdenoCA
6          &lt;NA&gt;    G3   Serous cystadenocarcinoma, NOS          Ovary-AdenoCA
  Histology.Tier1 Histology.Tier2             Histology.Tier3
1        ENDODERM         Bladder Transitional cell carcinoma
2        ECTODERM          Breast              Adenocarcinoma
3        ENDODERM         Bladder Transitional cell carcinoma
4        ECTODERM          Breast              Adenocarcinoma
5        MESODERM           Ovary              Adenocarcinoma
6        MESODERM           Ovary              Adenocarcinoma
                         Histology.Tier4 ICD.10.Classification ICGC.Sample.ID
1 Transitional cell carcinoma, papillary                 C67.9         SA4557
2            Infiltrating duct carcinoma                 C50.9        SA48667
3            Transitional cell carcinoma                 C67.9         SA4666
4            Infiltrating duct carcinoma                 C50.9        SA49610
5              Serous cystadenocarcinoma                 C48.2       SA505244
6              Serous cystadenocarcinoma                   C56       SA505270
  Mutation.Count Oncotree.Code    Organ.System Overall.Survival..Months.
1            250          BLSC URINARY BLADDER                  0.000000
2             61           IDC          BREAST                  0.000000
3            120          BLSC URINARY BLADDER                  0.000000
4             75           IDC          BREAST                  0.000000
5             62           SOC           OVARY                 15.690789
6             64           SOC           OVARY                  6.052632
  Overall.Survival.Status Ploidy Project.Code Purity Purity.Configuration
1                0:LIVING  3.228      BLCA-US  0.613                0.016
2                0:LIVING  3.986      BRCA-US  0.693                0.002
3                0:LIVING  2.002      BLCA-US  0.827                0.028
4                0:LIVING  3.359      BRCA-US  0.727                0.031
5              1:DECEASED  2.112        OV-AU  0.540                0.023
6              1:DECEASED  2.623        OV-AU  0.478                0.010
  Sample.Class Number.of.Samples.Per.Patient Sample.Type Sequencing.Type    Sex
1        Tumor                             1     Primary             WGS Female
2        Tumor                             1     Primary             WGS Female
3        Tumor                             1     Primary             WGS   Male
4        Tumor                             1     Primary             WGS Female
5        Tumor                             1     Primary             WGS Female
6        Tumor                             1     Primary             WGS Female
  Tumor.Stage..AJCC. TMB..nonsynonymous. Patient.Smoking.History.Category
1               &lt;NA&gt;            8.433333                             &lt;NA&gt;
2               &lt;NA&gt;            2.066667                             &lt;NA&gt;
3               &lt;NA&gt;            4.000000                             &lt;NA&gt;
4               &lt;NA&gt;            2.500000                             &lt;NA&gt;
5                III            2.066667                             &lt;NA&gt;
6                III            2.100000                             &lt;NA&gt;
  Tobacco.Smoking.Intensity Tumor.Sample.Histology.Code
1                        NA                      8130/3
2                        NA                      8500/3
3                        NA                      8120/3
4                        NA                      8500/3
5                        NA                      8441/3
6                        NA                      8441/3
  Whole.Genome.Duplication
1                      wgd
2                      wgd
3                   no_wgd
4                      wgd
5                   no_wgd
6                      wgd</code></pre>
<pre class="r"><code>summary(dat2)</code></pre>
<pre><code>              Study.ID      Patient.ID      Sample.ID    Age.at.Diagnosis
 pancan_pcawg_2020:2922   DO51954:  10   SP1003  :   1   Min.   : 1.00   
                          DO51964:   9   SP10084 :   1   1st Qu.:49.00   
                          DO52509:   6   SP1009  :   1   Median :60.00   
                          DO52510:   6   SP10150 :   1   Mean   :56.58   
                          DO52512:   6   SP101515:   1   3rd Qu.:70.00   
                          DO52513:   6   SP101519:   1   Max.   :90.00   
                          (Other):2879   (Other) :2916   NA&#39;s   :124     
                Alcohol    
 Don&#39;t know/Not sure: 989  
 No                 :  81  
 Yes                : 181  
 NA&#39;s               :1671  
                           
                           
                           
                                  Alcohol.History.Intensity Primary.Ancestry
 Daily Drinker                                 :  56        AFR : 124       
 None                                          :  81        AMR :  26       
 Not Documented                                :1006        ASN : 415       
 Occassional Drinker (&lt; once a month)          :  19        EUR :1968       
 Social Drinker (&gt; once a month, &lt; once a week):  49        SAN :  36       
 Weekly Drinker (&gt;=1x a week)                  :  40        NA&#39;s: 353       
 NA&#39;s                                          :1671                        
                 Cancer.Type  
 Pancreatic Cancer     : 381  
 Hepatobiliary Cancer  : 358  
 Prostate Cancer       : 275  
 Renal Cell Carcinoma  : 235  
 Breast Cancer         : 213  
 Esophagogastric Cancer: 165  
 (Other)               :1295  
                                              Cancer.Type.Detailed
 Hepatocellular Carcinoma                               : 315     
 Pancreatic Adenocarcinoma                              : 283     
 Prostate Adenocarcinoma                                : 275     
 Breast Invasive Ductal Carcinoma                       : 180     
 Renal Clear Cell Carcinoma                             : 160     
 Chronic Lymphocytic Leukemia/Small Lymphocytic Lymphoma: 158     
 (Other)                                                :1551     
  Cellularity             First.Therapy.Response           First.Therapy 
 Min.   : 12.00   Complete Response  : 199       No Treatment     : 405  
 1st Qu.: 71.00   Disease Progression:  39       Other Therapy    : 245  
 Median : 84.00   Partial Response   :  20       Chemotherapy     : 214  
 Mean   : 80.63   Stable Disease     :  64       Surgery          :  65  
 3rd Qu.: 95.00   Unknown            : 278       Radiation Therapy:  50  
 Max.   :100.00   NA&#39;s               :2322       (Other)          :  49  
 NA&#39;s   :2080                                    NA&#39;s             :1894  
                           Grade     
 G3                           :  75  
 G2                           :  52  
 II                           :  50  
 2 - Moderately differentiated:  37  
 G4                           :  37  
 (Other)                      : 148  
 NA&#39;s                         :2523  
                                                   Histology   
 Adenocarcinoma, NOS                                    : 467  
 Infiltrating duct carcinoma, NOS                       : 402  
 Hepatocellular carcinoma, NOS                          : 315  
 Clear cell adenocarcinoma, NOS                         : 160  
 Chronic lymphocytic leukemia/small lymphocytic lymphoma: 158  
 (Other)                                                :1344  
 NA&#39;s                                                   :  76  
    Histology.Abbreviation     Histology.Tier1 Histology.Tier2
 Liver-HCC     : 314       ECTODERM    : 211   Liver   : 314  
 Panc-AdenoCA  : 232       ENDODERM    :1232   Pancreas: 313  
 Prost-AdenoCA : 199       MESODERM    : 746   CNS     : 287  
 Breast-AdenoCA: 195       NEURAL_CREST: 394   Breast  : 211  
 Kidney-RCC    : 143       NA&#39;s        : 339   Prostate: 199  
 (Other)       :1500                           (Other) :1259  
 NA&#39;s          : 339                           NA&#39;s    : 339  
                                Histology.Tier3
 Adenocarcinoma                         :1118  
 Hepatocellular carcinoma               : 314  
 Renal cell carcinoma (proximal tubules): 143  
 Medulloblastoma                        : 141  
 Squamous cell carcinoma                : 121  
 (Other)                                : 746  
 NA&#39;s                                   : 339  
                    Histology.Tier4 ICD.10.Classification  ICGC.Sample.ID
 Adenocarcinoma             : 426   C61    : 275          SA100232:   1  
 Hepatocellular carcinoma   : 305   C22    : 260          SA100712:   1  
 Infiltrating duct carcinoma: 191   C25.0  : 212          SA100868:   1  
 Pancreatic ductal carcinoma: 157   C91.1  : 158          SA101371:   1  
 Serous cystadenocarcinoma  : 120   0      : 142          SA10195 :   1  
 (Other)                    :1384   C71    : 141          (Other) :2578  
 NA&#39;s                       : 339   (Other):1734          NA&#39;s    : 339  
 Mutation.Count     Oncotree.Code 
 Min.   :    1.00   HCC    : 323  
 1st Qu.:   21.25   PAAD   : 285  
 Median :   45.00   PRAD   : 275  
 Mean   :  105.39   IDC    : 180  
 3rd Qu.:   84.00   CCRCC  : 160  
 Max.   :12012.00   CLLSLL : 158  
 NA&#39;s   :252        (Other):1541  
                                                                Organ.System 
 LIVER                                                                : 336  
 PANCREAS                                                             : 313  
 BRAIN, &amp; CRANIAL NERVES, &amp; SPINAL CORD, (EXCL. VENTRICLE, CEREBELLUM): 287  
 BREAST                                                               : 211  
 PROSTATE GLAND                                                       : 199  
 (Other)                                                              :1237  
 NA&#39;s                                                                 : 339  
 Overall.Survival..Months. Overall.Survival.Status     Ploidy     
 Min.   : 0.00             0:LIVING  :1905         Min.   :1.283  
 1st Qu.:14.11             1:DECEASED: 890         1st Qu.:1.926  
 Median :29.14             NA&#39;s      : 127         Median :2.000  
 Mean   :32.95                                     Mean   :2.365  
 3rd Qu.:53.22                                     3rd Qu.:2.795  
 Max.   :94.70                                     Max.   :6.217  
 NA&#39;s   :2561                                      NA&#39;s   :219    
  Project.Code      Purity       Purity.Configuration    Sample.Class 
 LIRI-JP: 260   Min.   :0.1300   Min.   :0.0000       Cell line:   1  
 PBCA-DE: 230   1st Qu.:0.4730   1st Qu.:0.0060       Tumor    :2919  
 PACA-AU: 166   Median :0.6590   Median :0.0130       NA&#39;s     :   2  
 CLLE-ES: 158   Mean   :0.6401   Mean   :0.0211                       
 PACA-CA: 143   3rd Qu.:0.8240   3rd Qu.:0.0220                       
 RECA-EU: 123   Max.   :1.0000   Max.   :0.4180                       
 (Other):1842   NA&#39;s   :219      NA&#39;s   :219                          
 Number.of.Samples.Per.Patient           Sample.Type      Sequencing.Type
 Min.   : 1.000                Cell line       :   1   WGS        :2576  
 1st Qu.: 1.000                Local Recurrence:  41   WGS+RNA-Seq:   7  
 Median : 1.000                Metastasis      : 147   NA&#39;s       : 339  
 Mean   : 1.337                Primary         :2731                     
 3rd Qu.: 1.000                NA&#39;s            :   2                     
 Max.   :10.000                                                          
                                                                         
     Sex       Tumor.Stage..AJCC. TMB..nonsynonymous.
 Female:1285   unknown: 148       Min.   :  0.000    
 Male  :1637   2      : 119       1st Qu.:  0.700    
               III    : 104       Median :  1.500    
               3      :  75       Mean   :  3.507    
               A      :  66       3rd Qu.:  2.800    
               (Other):1112       Max.   :402.033    
               NA&#39;s   :1298       NA&#39;s   :239        
                                                     Patient.Smoking.History.Category
 Current reformed smoker for &lt;= 15 years                             :  21           
 Current reformed smoker for &gt; 15 years                              :  71           
 Current reformed smoker, duration not specified                     :  74           
 Current smoker (includes daily smokers non-daily/occasional smokers):  73           
 Lifelong non-smoker (&lt;100 cigarettes smoked in lifetime)            : 183           
 Smoking history not documented                                      : 829           
 NA&#39;s                                                                :1671           
 Tobacco.Smoking.Intensity Tumor.Sample.Histology.Code Whole.Genome.Duplication
 Min.   : 0.00             8140/3 : 430                no_wgd:1900             
 1st Qu.: 0.00             8500/3 : 348                wgd   : 803             
 Median :22.00             8170/3 : 305                NA&#39;s  : 219             
 Mean   :27.65             8070/3 : 120                                        
 3rd Qu.:50.00             8441/3 : 120                                        
 Max.   :86.00             (Other):1260                                        
 NA&#39;s   :2775              NA&#39;s   : 339                                        </code></pre>
<pre class="r"><code># A basic box plot
ggplot(dat2, aes(x=Sex, y=Age.at.Diagnosis)) + geom_boxplot()</code></pre>
<pre><code>Warning: Removed 124 rows containing non-finite values (stat_boxplot).</code></pre>
<p><img src="figure/R-output.Rmd/unnamed-chunk-27-1.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code># A basic box with the conditions colored
ggplot(dat2, aes(x=Sex, y=Age.at.Diagnosis,fill=Sex)) + geom_boxplot()</code></pre>
<pre><code>Warning: Removed 124 rows containing non-finite values (stat_boxplot).</code></pre>
<p><img src="figure/R-output.Rmd/unnamed-chunk-28-1.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>ggplot(dat2, aes(x=Sex, y=Age.at.Diagnosis,fill=Sex)) + geom_boxplot()+guides(fill=FALSE)</code></pre>
<pre><code>Warning: `guides(&lt;scale&gt; = FALSE)` is deprecated. Please use `guides(&lt;scale&gt; =
&quot;none&quot;)` instead.</code></pre>
<pre><code>Warning: Removed 124 rows containing non-finite values (stat_boxplot).</code></pre>
<p><img src="figure/R-output.Rmd/unnamed-chunk-29-1.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>ggplot(dat2, aes(x=Sex, y=Age.at.Diagnosis,fill=Sex)) + geom_boxplot()+guides(fill=FALSE)+ coord_flip()</code></pre>
<pre><code>Warning: `guides(&lt;scale&gt; = FALSE)` is deprecated. Please use `guides(&lt;scale&gt; =
&quot;none&quot;)` instead.</code></pre>
<pre><code>Warning: Removed 124 rows containing non-finite values (stat_boxplot).</code></pre>
<p><img src="figure/R-output.Rmd/unnamed-chunk-30-1.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>ggplot(dat2, aes(x=Sex, y=Age.at.Diagnosis)) + geom_boxplot()+stat_summary(fun.y=mean,geom=&quot;point&quot;,na.rm=T,shape=5,size=4)</code></pre>
<pre><code>Warning: `fun.y` is deprecated. Use `fun` instead.</code></pre>
<pre><code>Warning: Removed 124 rows containing non-finite values (stat_boxplot).</code></pre>
<p><img src="figure/R-output.Rmd/unnamed-chunk-31-1.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code># Basic line graph
ggplot(data=dat2, aes(x=Sex, y=Age.at.Diagnosis, group=1)) +
    geom_line()</code></pre>
<p><img src="figure/R-output.Rmd/unnamed-chunk-32-1.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code># Add points
ggplot(data=dat2, aes(x=Sex, y=Age.at.Diagnosis)) +
    geom_line(aes(group=1))+geom_point()</code></pre>
<pre><code>Warning: Removed 124 rows containing missing values (geom_point).</code></pre>
<p><img src="figure/R-output.Rmd/unnamed-chunk-33-1.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code># Change color of line 
# Change line type  and use thicker line 
ggplot(data=dat2, aes(x=Sex, y=Age.at.Diagnosis)) +
    geom_line(colour=&quot;red&quot;, linetype=&quot;dashed&quot;, size=1.5,aes(group=1))</code></pre>
<p><img src="figure/R-output.Rmd/unnamed-chunk-34-1.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code># Change the y-range to go from 0 to the maximum value in the Age.at.Diagnosis,
# and change axis labels
ggplot(data=dat2, aes(x=Sex, y=Age.at.Diagnosis)) +
    geom_line(aes(group=1))+expand_limits(y=0) +
    xlab(&quot;Gender of patients&quot;) + ylab(&quot;Age at Diagnosis&quot;) +
    ggtitle(&quot;Average age of diagnosis&quot;)</code></pre>
<p><img src="figure/R-output.Rmd/unnamed-chunk-35-1.png" width="672" style="display: block; margin: auto;" /></p>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span>
Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<pre class="r"><code>sessionInfo()</code></pre>
<pre><code>R version 4.1.3 (2022-03-10)
Platform: x86_64-w64-mingw32/x64 (64-bit)
Running under: Windows 10 x64 (build 19044)

Matrix products: default

locale:
[1] LC_COLLATE=English_United States.1252 
[2] LC_CTYPE=English_United States.1252   
[3] LC_MONETARY=English_United States.1252
[4] LC_NUMERIC=C                          
[5] LC_TIME=English_United States.1252    

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
[1] ggplot2_3.3.5   workflowr_1.7.0

loaded via a namespace (and not attached):
 [1] tidyselect_1.1.2 xfun_0.30        bslib_0.3.1      purrr_0.3.4     
 [5] colorspace_2.0-3 vctrs_0.4.0      generics_0.1.2   htmltools_0.5.2 
 [9] yaml_2.3.5       utf8_1.2.2       rlang_1.0.2      jquerylib_0.1.4 
[13] later_1.3.0      pillar_1.7.0     glue_1.6.2       withr_2.5.0     
[17] lifecycle_1.0.1  stringr_1.4.0    munsell_0.5.0    gtable_0.3.0    
[21] evaluate_0.15    labeling_0.4.2   knitr_1.40       callr_3.7.0     
[25] fastmap_1.1.0    httpuv_1.6.5     ps_1.6.0         fansi_1.0.3     
[29] highr_0.9        Rcpp_1.0.8.3     promises_1.2.0.1 scales_1.1.1    
[33] jsonlite_1.8.0   farver_2.1.0     fs_1.5.2         digest_0.6.29   
[37] stringi_1.7.6    processx_3.5.3   dplyr_1.0.8      getPass_0.2-2   
[41] rprojroot_2.0.3  grid_4.1.3       cli_3.2.0        tools_4.1.3     
[45] magrittr_2.0.3   sass_0.4.1       tibble_3.1.6     crayon_1.5.1    
[49] whisker_0.4      pkgconfig_2.0.3  ellipsis_0.3.2   rmarkdown_2.13  
[53] httr_1.4.2       rstudioapi_0.13  R6_2.5.1         git2r_0.30.1    
[57] compiler_4.1.3  </code></pre>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
https://docs.mathjax.org/en/latest/web/configuration.html. This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>




</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->

<script>
$(document).ready(function ()  {

    // temporarily add toc-ignore selector to headers for the consistency with Pandoc
    $('.unlisted.unnumbered').addClass('toc-ignore')

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_');
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
